[gadi-cpu-clx-2095:102047] *** Process received signal ***
[gadi-cpu-clx-2095:102047] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102047] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102047] Failing at address: 0x15033be19008
[gadi-cpu-clx-2095:102047] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x15035b83fdd0]
[gadi-cpu-clx-2095:102047] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x1503366c00b9]
[gadi-cpu-clx-2095:102047] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x1503364b62a3]
[gadi-cpu-clx-2095:102047] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x1503366bfd54]
[gadi-cpu-clx-2095:102047] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x15033780b61e]
[gadi-cpu-clx-2095:102047] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x1503377c2a5d]
[gadi-cpu-clx-2095:102047] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x15033777f3a4]
[gadi-cpu-clx-2095:102047] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x150337a5b7fb]
[gadi-cpu-clx-2095:102047] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x15033808998e]
[gadi-cpu-clx-2095:102047] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x150339de42da]
[gadi-cpu-clx-2095:102047] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x15033a0d34b2]
[gadi-cpu-clx-2095:102047] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x15033a0f4d79]
[gadi-cpu-clx-2095:102047] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x15033a3b6a5c]
[gadi-cpu-clx-2095:102047] [13] python3(PyModule_ExecDef+0x78)[0x556d65a1a9d8]
[gadi-cpu-clx-2095:102047] [14] python3(+0x1f2a45)[0x556d65a1aa45]
[gadi-cpu-clx-2095:102047] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x556d659715da]
[gadi-cpu-clx-2095:102047] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x556d659f045a]
[gadi-cpu-clx-2095:102047] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x556d65941505]
[gadi-cpu-clx-2095:102047] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x556d65961133]
[gadi-cpu-clx-2095:102047] [19] python3(+0x181455)[0x556d659a9455]
[gadi-cpu-clx-2095:102047] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x556d659ef762]
[gadi-cpu-clx-2095:102047] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x556d65960d37]
[gadi-cpu-clx-2095:102047] [22] python3(+0x181455)[0x556d659a9455]
[gadi-cpu-clx-2095:102047] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x556d659eb4d1]
[gadi-cpu-clx-2095:102047] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x556d65960d37]
[gadi-cpu-clx-2095:102047] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x556d659eb2b5]
[gadi-cpu-clx-2095:102047] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x556d65960d37]
[gadi-cpu-clx-2095:102047] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x556d659eb2b5]
[gadi-cpu-clx-2095:102047] [28] python3(_PyObject_FastCallDict+0x1b6)[0x556d65942ab6]
[gadi-cpu-clx-2095:102047] [29] python3(+0x11b9a2)[0x556d659439a2]
[gadi-cpu-clx-2095:102047] *** End of error message ***
--------------------------------------------------------------------------
Primary job  terminated normally, but 1 process returned
a non-zero exit code. Per user-direction, the job has been aborted.
--------------------------------------------------------------------------
[gadi-cpu-clx-2095:102055] *** Process received signal ***
[gadi-cpu-clx-2095:102055] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102055] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102055] Failing at address: 0x14df22582008
[gadi-cpu-clx-2095:102055] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x14df41fa8dd0]
[gadi-cpu-clx-2095:102055] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x14df1ce290b9]
[gadi-cpu-clx-2095:102055] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x14df1cc1f2a3]
[gadi-cpu-clx-2095:102055] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x14df1ce28d54]
[gadi-cpu-clx-2095:102055] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x14df1df7461e]
[gadi-cpu-clx-2095:102055] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x14df1df2ba5d]
[gadi-cpu-clx-2095:102055] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x14df1dee83a4]
[gadi-cpu-clx-2095:102055] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x14df1e1c47fb]
[gadi-cpu-clx-2095:102055] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x14df1e7f298e]
[gadi-cpu-clx-2095:102055] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x14df2058d2da]
[gadi-cpu-clx-2095:102055] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x14df2087c4b2]
[gadi-cpu-clx-2095:102055] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x14df2089dd79]
[gadi-cpu-clx-2095:102055] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x14df20b5fa5c]
[gadi-cpu-clx-2095:102055] [13] python3(PyModule_ExecDef+0x78)[0x561227e3e9d8]
[gadi-cpu-clx-2095:102055] [14] python3(+0x1f2a45)[0x561227e3ea45]
[gadi-cpu-clx-2095:102055] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x561227d955da]
[gadi-cpu-clx-2095:102055] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x561227e1445a]
[gadi-cpu-clx-2095:102055] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x561227d65505]
[gadi-cpu-clx-2095:102055] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x561227d85133]
[gadi-cpu-clx-2095:102055] [19] python3(+0x181455)[0x561227dcd455]
[gadi-cpu-clx-2095:102055] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x561227e13762]
[gadi-cpu-clx-2095:102055] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x561227d84d37]
[gadi-cpu-clx-2095:102055] [22] python3(+0x181455)[0x561227dcd455]
[gadi-cpu-clx-2095:102055] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x561227e0f4d1]
[gadi-cpu-clx-2095:102055] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x561227d84d37]
[gadi-cpu-clx-2095:102055] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x561227e0f2b5]
[gadi-cpu-clx-2095:102055] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x561227d84d37]
[gadi-cpu-clx-2095:102055] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x561227e0f2b5]
[gadi-cpu-clx-2095:102055] [28] python3(_PyObject_FastCallDict+0x1b6)[0x561227d66ab6]
[gadi-cpu-clx-2095:102055] [29] python3(+0x11b9a2)[0x561227d679a2]
[gadi-cpu-clx-2095:102055] *** End of error message ***
[gadi-cpu-clx-2095:102073] *** Process received signal ***
[gadi-cpu-clx-2095:102073] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102073] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102073] Failing at address: 0x1458d98b1008
[gadi-cpu-clx-2095:102075] *** Process received signal ***
[gadi-cpu-clx-2095:102075] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102075] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102075] Failing at address: 0x152ee84bb008
[gadi-cpu-clx-2095:102075] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x152f07ee1dd0]
[gadi-cpu-clx-2095:102075] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x152edec760b9]
[gadi-cpu-clx-2095:102075] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x152edea6c2a3]
[gadi-cpu-clx-2095:102075] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x152edec75d54]
[gadi-cpu-clx-2095:102075] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x152edfdc161e]
[gadi-cpu-clx-2095:102075] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x152edfd78a5d]
[gadi-cpu-clx-2095:102075] [ 6] [gadi-cpu-clx-2095:102032] *** Process received signal ***
[gadi-cpu-clx-2095:102032] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102032] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102032] Failing at address: 0x14a615898008
[gadi-cpu-clx-2095:102032] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x14a6352bedd0]
[gadi-cpu-clx-2095:102032] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x14a61013f0b9]
[gadi-cpu-clx-2095:102032] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x14a60bdf82a3]
[gadi-cpu-clx-2095:102032] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x14a61013ed54]
[gadi-cpu-clx-2095:102032] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x14a61128a61e]
[gadi-cpu-clx-2095:102032] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x14a611241a5d]
[gadi-cpu-clx-2095:102032] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x14a6111fe3a4]
[gadi-cpu-clx-2095:102032] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x14a6114da7fb]
[gadi-cpu-clx-2095:102032] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x14a611b0898e]
[gadi-cpu-clx-2095:102032] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x14a6138632da]
[gadi-cpu-clx-2095:102032] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x14a613b524b2]
[gadi-cpu-clx-2095:102032] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x14a613b73d79]
[gadi-cpu-clx-2095:102032] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x14a613e35a5c]
[gadi-cpu-clx-2095:102032] [13] python3(PyModule_ExecDef+0x78)[0x56403aa309d8]
[gadi-cpu-clx-2095:102032] [14] python3(+0x1f2a45)[0x56403aa30a45]
[gadi-cpu-clx-2095:102032] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x56403a9875da]
[gadi-cpu-clx-2095:102032] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x56403aa0645a]
[gadi-cpu-clx-2095:102032] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x56403a957505]
[gadi-cpu-clx-2095:102032] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x56403a977133]
[gadi-cpu-clx-2095:102032] [19] python3(+0x181455)[0x56403a9bf455]
[gadi-cpu-clx-2095:102032] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x56403aa05762]
[gadi-cpu-clx-2095:102032] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x56403a976d37]
[gadi-cpu-clx-2095:102032] [22] python3(+0x181455)[0x56403a9bf455]
[gadi-cpu-clx-2095:102032] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x56403aa014d1]
[gadi-cpu-clx-2095:102032] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x56403a976d37]
[gadi-cpu-clx-2095:102032] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x56403aa012b5]
[gadi-cpu-clx-2095:102032] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x56403a976d37]
[gadi-cpu-clx-2095:102032] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x56403aa012b5]
[gadi-cpu-clx-2095:102032] [28] python3(_PyObject_FastCallDict+0x1b6)[0x56403a958ab6]
[gadi-cpu-clx-2095:102032] [29] python3(+0x11b9a2)[0x56403a9599a2]
[gadi-cpu-clx-2095:102032] *** End of error message ***
[gadi-cpu-clx-2095:102031] *** Process received signal ***
[gadi-cpu-clx-2095:102031] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102031] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102031] Failing at address: 0x1498a7884008
[gadi-cpu-clx-2095:102031] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x1498c72aadd0]
[gadi-cpu-clx-2095:102031] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x1498a212b0b9]
[gadi-cpu-clx-2095:102031] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x1498a1f212a3]
[gadi-cpu-clx-2095:102031] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x1498a212ad54]
[gadi-cpu-clx-2095:102031] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x1498a327661e]
[gadi-cpu-clx-2095:102031] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x1498a322da5d]
[gadi-cpu-clx-2095:102031] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x1498a31ea3a4]
[gadi-cpu-clx-2095:102031] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x1498a34c67fb]
[gadi-cpu-clx-2095:102031] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x1498a3af498e]
[gadi-cpu-clx-2095:102031] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x1498a588f2da]
[gadi-cpu-clx-2095:102031] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x1498a5b7e4b2]
[gadi-cpu-clx-2095:102031] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x1498a5b9fd79]
[gadi-cpu-clx-2095:102031] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x1498a5e61a5c]
[gadi-cpu-clx-2095:102031] [13] python3(PyModule_ExecDef+0x78)[0x559faa26d9d8]
[gadi-cpu-clx-2095:102031] [14] python3(+0x1f2a45)[0x559faa26da45]
[gadi-cpu-clx-2095:102031] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x559faa1c45da]
[gadi-cpu-clx-2095:102031] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x559faa24345a]
[gadi-cpu-clx-2095:102031] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x559faa194505]
[gadi-cpu-clx-2095:102031] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x559faa1b4133]
[gadi-cpu-clx-2095:102031] [19] python3(+0x181455)[0x559faa1fc455]
[gadi-cpu-clx-2095:102031] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x559faa242762]
[gadi-cpu-clx-2095:102031] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x559faa1b3d37]
[gadi-cpu-clx-2095:102031] [22] python3(+0x181455)[0x559faa1fc455]
[gadi-cpu-clx-2095:102031] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x559faa23e4d1]
[gadi-cpu-clx-2095:102031] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x559faa1b3d37]
[gadi-cpu-clx-2095:102031] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x559faa23e2b5]
[gadi-cpu-clx-2095:102031] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x559faa1b3d37]
[gadi-cpu-clx-2095:102031] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x559faa23e2b5]
[gadi-cpu-clx-2095:102031] [28] python3(_PyObject_FastCallDict+0x1b6)[0x559faa195ab6]
[gadi-cpu-clx-2095:102031] [29] python3(+0x11b9a2)[0x559faa1969a2]
[gadi-cpu-clx-2095:102031] *** End of error message ***
[gadi-cpu-clx-2095:102033] *** Process received signal ***
[gadi-cpu-clx-2095:102033] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102033] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102033] Failing at address: 0x14802671b008
[gadi-cpu-clx-2095:102033] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x148046141dd0]
[gadi-cpu-clx-2095:102033] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x148020fc20b9]
[gadi-cpu-clx-2095:102033] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x148020db82a3]
[gadi-cpu-clx-2095:102033] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x148020fc1d54]
[gadi-cpu-clx-2095:102033] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x14802210d61e]
[gadi-cpu-clx-2095:102033] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x1480220c4a5d]
[gadi-cpu-clx-2095:102033] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x1480220813a4]
[gadi-cpu-clx-2095:102033] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x14802235d7fb]
[gadi-cpu-clx-2095:102033] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x14802298b98e]
[gadi-cpu-clx-2095:102033] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x1480246e62da]
[gadi-cpu-clx-2095:102033] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x1480249d54b2]
[gadi-cpu-clx-2095:102033] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x1480249f6d79]
[gadi-cpu-clx-2095:102033] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x148024cb8a5c]
[gadi-cpu-clx-2095:102033] [13] python3(PyModule_ExecDef+0x78)[0x561a3bec69d8]
[gadi-cpu-clx-2095:102033] [14] python3(+0x1f2a45)[0x561a3bec6a45]
[gadi-cpu-clx-2095:102033] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x561a3be1d5da]
[gadi-cpu-clx-2095:102033] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x561a3be9c45a]
[gadi-cpu-clx-2095:102033] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x561a3bded505]
[gadi-cpu-clx-2095:102033] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x561a3be0d133]
[gadi-cpu-clx-2095:102033] [19] python3(+0x181455)[0x561a3be55455]
[gadi-cpu-clx-2095:102033] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x561a3be9b762]
[gadi-cpu-clx-2095:102033] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x561a3be0cd37]
[gadi-cpu-clx-2095:102033] [22] python3(+0x181455)[0x561a3be55455]
[gadi-cpu-clx-2095:102033] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x561a3be974d1]
[gadi-cpu-clx-2095:102033] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x561a3be0cd37]
[gadi-cpu-clx-2095:102033] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x561a3be972b5]
[gadi-cpu-clx-2095:102033] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x561a3be0cd37]
[gadi-cpu-clx-2095:102033] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x561a3be972b5]
[gadi-cpu-clx-2095:102033] [28] python3(_PyObject_FastCallDict+0x1b6)[0x561a3bdeeab6]
[gadi-cpu-clx-2095:102033] [29] python3(+0x11b9a2)[0x561a3bdef9a2]
[gadi-cpu-clx-2095:102033] *** End of error message ***
[gadi-cpu-clx-2095:102030] *** Process received signal ***
[gadi-cpu-clx-2095:102030] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102030] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102030] Failing at address: 0x147ab7d78008
[gadi-cpu-clx-2095:102030] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x147ad779edd0]
[gadi-cpu-clx-2095:102030] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x147ab261f0b9]
[gadi-cpu-clx-2095:102030] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x147ab24152a3]
[gadi-cpu-clx-2095:102030] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x147ab261ed54]
[gadi-cpu-clx-2095:102030] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x147ab376a61e]
[gadi-cpu-clx-2095:102030] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x147ab3721a5d]
[gadi-cpu-clx-2095:102030] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x147ab36de3a4]
[gadi-cpu-clx-2095:102030] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x147ab39ba7fb]
[gadi-cpu-clx-2095:102030] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x147ab3fe898e]
[gadi-cpu-clx-2095:102030] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x147ab5d432da]
[gadi-cpu-clx-2095:102030] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x147ab60324b2]
[gadi-cpu-clx-2095:102030] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x147ab6053d79]
[gadi-cpu-clx-2095:102030] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x147ab6315a5c]
[gadi-cpu-clx-2095:102030] [13] python3(PyModule_ExecDef+0x78)[0x55fb3acec9d8]
[gadi-cpu-clx-2095:102030] [14] python3(+0x1f2a45)[0x55fb3aceca45]
[gadi-cpu-clx-2095:102030] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x55fb3ac435da]
[gadi-cpu-clx-2095:102030] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x55fb3acc245a]
[gadi-cpu-clx-2095:102030] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x55fb3ac13505]
[gadi-cpu-clx-2095:102030] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x55fb3ac33133]
[gadi-cpu-clx-2095:102030] [19] python3(+0x181455)[0x55fb3ac7b455]
[gadi-cpu-clx-2095:102030] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x55fb3acc1762]
[gadi-cpu-clx-2095:102030] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x55fb3ac32d37]
[gadi-cpu-clx-2095:102030] [22] python3(+0x181455)[0x55fb3ac7b455]
[gadi-cpu-clx-2095:102030] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x55fb3acbd4d1]
[gadi-cpu-clx-2095:102030] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x55fb3ac32d37]
[gadi-cpu-clx-2095:102030] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x55fb3acbd2b5]
[gadi-cpu-clx-2095:102030] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x55fb3ac32d37]
[gadi-cpu-clx-2095:102030] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x55fb3acbd2b5]
[gadi-cpu-clx-2095:102030] [28] python3(_PyObject_FastCallDict+0x1b6)[0x55fb3ac14ab6]
[gadi-cpu-clx-2095:102030] [29] python3(+0x11b9a2)[0x55fb3ac159a2]
[gadi-cpu-clx-2095:102030] *** End of error message ***
[gadi-cpu-clx-2095:102036] *** Process received signal ***
[gadi-cpu-clx-2095:102036] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102036] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102036] Failing at address: 0x14addd3a4008
[gadi-cpu-clx-2095:102036] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x14adfcdcadd0]
[gadi-cpu-clx-2095:102036] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x14add3beb0b9]
[gadi-cpu-clx-2095:102036] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x14add39e12a3]
[gadi-cpu-clx-2095:102036] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x14add3bead54]
[gadi-cpu-clx-2095:102036] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x14add8d9661e]
[gadi-cpu-clx-2095:102036] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x14add8d4da5d]
[gadi-cpu-clx-2095:102036] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x14add8d0a3a4]
[gadi-cpu-clx-2095:102036] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x14add8fe67fb]
[gadi-cpu-clx-2095:102036] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x14add961498e]
[gadi-cpu-clx-2095:102036] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x14addb36f2da]
[gadi-cpu-clx-2095:102036] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x14addb65e4b2]
[gadi-cpu-clx-2095:102036] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x14addb67fd79]
[gadi-cpu-clx-2095:102036] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x14addb941a5c]
[gadi-cpu-clx-2095:102036] [13] python3(PyModule_ExecDef+0x78)[0x55c47e5249d8]
[gadi-cpu-clx-2095:102036] [14] python3(+0x1f2a45)[0x55c47e524a45]
[gadi-cpu-clx-2095:102036] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x55c47e47b5da]
[gadi-cpu-clx-2095:102036] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x55c47e4fa45a]
[gadi-cpu-clx-2095:102036] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x55c47e44b505]
[gadi-cpu-clx-2095:102036] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x55c47e46b133]
[gadi-cpu-clx-2095:102036] [19] python3(+0x181455)[0x55c47e4b3455]
[gadi-cpu-clx-2095:102036] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x55c47e4f9762]
[gadi-cpu-clx-2095:102036] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x55c47e46ad37]
[gadi-cpu-clx-2095:102036] [22] python3(+0x181455)[0x55c47e4b3455]
[gadi-cpu-clx-2095:102036] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x55c47e4f54d1]
[gadi-cpu-clx-2095:102036] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x55c47e46ad37]
[gadi-cpu-clx-2095:102036] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x55c47e4f52b5]
[gadi-cpu-clx-2095:102036] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x55c47e46ad37]
[gadi-cpu-clx-2095:102036] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x55c47e4f52b5]
[gadi-cpu-clx-2095:102036] [28] python3(_PyObject_FastCallDict+0x1b6)[0x55c47e44cab6]
[gadi-cpu-clx-2095:102036] [29] python3(+0x11b9a2)[0x55c47e44d9a2]
[gadi-cpu-clx-2095:102036] *** End of error message ***
[gadi-cpu-clx-2095:102034] *** Process received signal ***
[gadi-cpu-clx-2095:102034] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102034] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102034] Failing at address: 0x14f96e263008
[gadi-cpu-clx-2095:102034] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x14f98dc89dd0]
[gadi-cpu-clx-2095:102034] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x14f968b0a0b9]
[gadi-cpu-clx-2095:102034] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x14f9689002a3]
[gadi-cpu-clx-2095:102034] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x14f968b09d54]
[gadi-cpu-clx-2095:102034] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x14f969c5561e]
[gadi-cpu-clx-2095:102034] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x14f969c0ca5d]
[gadi-cpu-clx-2095:102034] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x14f969bc93a4]
[gadi-cpu-clx-2095:102034] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x14f969ea57fb]
[gadi-cpu-clx-2095:102034] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x14f96a4d398e]
[gadi-cpu-clx-2095:102034] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x14f96c22e2da]
[gadi-cpu-clx-2095:102034] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x14f96c51d4b2]
[gadi-cpu-clx-2095:102034] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x14f96c53ed79]
[gadi-cpu-clx-2095:102034] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x14f96c800a5c]
[gadi-cpu-clx-2095:102034] [13] python3(PyModule_ExecDef+0x78)[0x56040acea9d8]
[gadi-cpu-clx-2095:102034] [14] python3(+0x1f2a45)[0x56040aceaa45]
[gadi-cpu-clx-2095:102034] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x56040ac415da]
[gadi-cpu-clx-2095:102034] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x56040acc045a]
[gadi-cpu-clx-2095:102034] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x56040ac11505]
[gadi-cpu-clx-2095:102034] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x56040ac31133]
[gadi-cpu-clx-2095:102034] [19] python3(+0x181455)[0x56040ac79455]
[gadi-cpu-clx-2095:102034] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x56040acbf762]
[gadi-cpu-clx-2095:102034] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x56040ac30d37]
[gadi-cpu-clx-2095:102034] [22] python3(+0x181455)[0x56040ac79455]
[gadi-cpu-clx-2095:102034] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x56040acbb4d1]
[gadi-cpu-clx-2095:102034] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x56040ac30d37]
[gadi-cpu-clx-2095:102034] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x56040acbb2b5]
[gadi-cpu-clx-2095:102034] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x56040ac30d37]
[gadi-cpu-clx-2095:102034] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x56040acbb2b5]
[gadi-cpu-clx-2095:102034] [28] python3(_PyObject_FastCallDict+0x1b6)[0x56040ac12ab6]
[gadi-cpu-clx-2095:102034] [29] python3(+0x11b9a2)[0x56040ac139a2]
[gadi-cpu-clx-2095:102034] *** End of error message ***
[gadi-cpu-clx-2095:102037] *** Process received signal ***
[gadi-cpu-clx-2095:102037] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102037] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102037] Failing at address: 0x151c307a4008
[gadi-cpu-clx-2095:102037] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x151c501cadd0]
[gadi-cpu-clx-2095:102037] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x151c26f900b9]
[gadi-cpu-clx-2095:102037] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x151c26d862a3]
[gadi-cpu-clx-2095:102037] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x151c26f8fd54]
[gadi-cpu-clx-2095:102037] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x151c2c19661e]
[gadi-cpu-clx-2095:102037] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x151c2c14da5d]
[gadi-cpu-clx-2095:102037] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x151c2c10a3a4]
[gadi-cpu-clx-2095:102037] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x151c2c3e67fb]
[gadi-cpu-clx-2095:102037] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x151c2ca1498e]
[gadi-cpu-clx-2095:102037] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x151c2e76f2da]
[gadi-cpu-clx-2095:102037] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x151c2ea5e4b2]
[gadi-cpu-clx-2095:102037] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x151c2ea7fd79]
[gadi-cpu-clx-2095:102037] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x151c2ed41a5c]
[gadi-cpu-clx-2095:102037] [13] python3(PyModule_ExecDef+0x78)[0x55bc757dc9d8]
[gadi-cpu-clx-2095:102037] [14] python3(+0x1f2a45)[0x55bc757dca45]
[gadi-cpu-clx-2095:102037] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x55bc757335da]
[gadi-cpu-clx-2095:102037] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x55bc757b245a]
[gadi-cpu-clx-2095:102037] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x55bc75703505]
[gadi-cpu-clx-2095:102037] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x55bc75723133]
[gadi-cpu-clx-2095:102037] [19] python3(+0x181455)[0x55bc7576b455]
[gadi-cpu-clx-2095:102037] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x55bc757b1762]
[gadi-cpu-clx-2095:102037] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x55bc75722d37]
[gadi-cpu-clx-2095:102037] [22] python3(+0x181455)[0x55bc7576b455]
[gadi-cpu-clx-2095:102037] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x55bc757ad4d1]
[gadi-cpu-clx-2095:102037] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x55bc75722d37]
[gadi-cpu-clx-2095:102037] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x55bc757ad2b5]
[gadi-cpu-clx-2095:102037] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x55bc75722d37]
[gadi-cpu-clx-2095:102037] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x55bc757ad2b5]
[gadi-cpu-clx-2095:102037] [28] python3(_PyObject_FastCallDict+0x1b6)[0x55bc75704ab6]
[gadi-cpu-clx-2095:102037] [29] python3(+0x11b9a2)[0x55bc757059a2]
[gadi-cpu-clx-2095:102037] *** End of error message ***
[gadi-cpu-clx-2095:102035] *** Process received signal ***
[gadi-cpu-clx-2095:102035] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102035] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102035] Failing at address: 0x152f91c80008
[gadi-cpu-clx-2095:102035] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x152fb16a6dd0]
[gadi-cpu-clx-2095:102035] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x152f8c5270b9]
[gadi-cpu-clx-2095:102035] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x152f8c31d2a3]
[gadi-cpu-clx-2095:102035] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x152f8c526d54]
[gadi-cpu-clx-2095:102035] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x152f8d67261e]
[gadi-cpu-clx-2095:102035] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x152f8d629a5d]
[gadi-cpu-clx-2095:102035] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x152f8d5e63a4]
[gadi-cpu-clx-2095:102035] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x152f8d8c27fb]
[gadi-cpu-clx-2095:102035] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x152f8def098e]
[gadi-cpu-clx-2095:102035] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x152f8fc4b2da]
[gadi-cpu-clx-2095:102035] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x152f8ff3a4b2]
[gadi-cpu-clx-2095:102035] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x152f8ff5bd79]
[gadi-cpu-clx-2095:102035] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x152f9021da5c]
[gadi-cpu-clx-2095:102035] [13] python3(PyModule_ExecDef+0x78)[0x556f173fa9d8]
[gadi-cpu-clx-2095:102035] [14] python3(+0x1f2a45)[0x556f173faa45]
[gadi-cpu-clx-2095:102035] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x556f173515da]
[gadi-cpu-clx-2095:102035] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x556f173d045a]
[gadi-cpu-clx-2095:102035] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x556f17321505]
[gadi-cpu-clx-2095:102035] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x556f17341133]
[gadi-cpu-clx-2095:102035] [19] python3(+0x181455)[0x556f17389455]
[gadi-cpu-clx-2095:102035] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x556f173cf762]
[gadi-cpu-clx-2095:102035] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x556f17340d37]
[gadi-cpu-clx-2095:102035] [22] python3(+0x181455)[0x556f17389455]
[gadi-cpu-clx-2095:102035] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x556f173cb4d1]
[gadi-cpu-clx-2095:102035] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x556f17340d37]
[gadi-cpu-clx-2095:102035] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x556f173cb2b5]
[gadi-cpu-clx-2095:102035] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x556f17340d37]
[gadi-cpu-clx-2095:102035] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x556f173cb2b5]
[gadi-cpu-clx-2095:102035] [28] python3(_PyObject_FastCallDict+0x1b6)[0x556f17322ab6]
[gadi-cpu-clx-2095:102035] [29] python3(+0x11b9a2)[0x556f173239a2]
[gadi-cpu-clx-2095:102035] *** End of error message ***
[gadi-cpu-clx-2095:102040] *** Process received signal ***
[gadi-cpu-clx-2095:102040] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102040] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102040] Failing at address: 0x14ad85f63008
[gadi-cpu-clx-2095:102040] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x14ada5989dd0]
[gadi-cpu-clx-2095:102040] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x14ad8080a0b9]
[gadi-cpu-clx-2095:102040] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x14ad806002a3]
[gadi-cpu-clx-2095:102040] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x14ad80809d54]
[gadi-cpu-clx-2095:102040] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x14ad8195561e]
[gadi-cpu-clx-2095:102040] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x14ad8190ca5d]
[gadi-cpu-clx-2095:102040] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x14ad818c93a4]
[gadi-cpu-clx-2095:102040] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x14ad81ba57fb]
[gadi-cpu-clx-2095:102040] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x14ad821d398e]
[gadi-cpu-clx-2095:102040] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x14ad83f2e2da]
[gadi-cpu-clx-2095:102040] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x14ad8421d4b2]
[gadi-cpu-clx-2095:102040] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x14ad8423ed79]
[gadi-cpu-clx-2095:102040] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x14ad84500a5c]
[gadi-cpu-clx-2095:102040] [13] python3(PyModule_ExecDef+0x78)[0x55705ee509d8]
[gadi-cpu-clx-2095:102040] [14] python3(+0x1f2a45)[0x55705ee50a45]
[gadi-cpu-clx-2095:102040] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x55705eda75da]
[gadi-cpu-clx-2095:102040] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x55705ee2645a]
[gadi-cpu-clx-2095:102040] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x55705ed77505]
[gadi-cpu-clx-2095:102040] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x55705ed97133]
[gadi-cpu-clx-2095:102040] [19] python3(+0x181455)[0x55705eddf455]
[gadi-cpu-clx-2095:102040] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x55705ee25762]
[gadi-cpu-clx-2095:102040] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x55705ed96d37]
[gadi-cpu-clx-2095:102040] [22] python3(+0x181455)[0x55705eddf455]
[gadi-cpu-clx-2095:102040] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x55705ee214d1]
[gadi-cpu-clx-2095:102040] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x55705ed96d37]
[gadi-cpu-clx-2095:102040] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x55705ee212b5]
[gadi-cpu-clx-2095:102040] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x55705ed96d37]
[gadi-cpu-clx-2095:102040] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x55705ee212b5]
[gadi-cpu-clx-2095:102040] [28] python3(_PyObject_FastCallDict+0x1b6)[0x55705ed78ab6]
[gadi-cpu-clx-2095:102040] [29] python3(+0x11b9a2)[0x55705ed799a2]
[gadi-cpu-clx-2095:102040] *** End of error message ***
[gadi-cpu-clx-2095:102038] *** Process received signal ***
[gadi-cpu-clx-2095:102038] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102038] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102038] Failing at address: 0x150e5abbe008
[gadi-cpu-clx-2095:102038] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x150e7a5e4dd0]
[gadi-cpu-clx-2095:102038] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x150e554650b9]
[gadi-cpu-clx-2095:102038] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x150e5525b2a3]
[gadi-cpu-clx-2095:102038] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x150e55464d54]
[gadi-cpu-clx-2095:102038] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x150e565b061e]
[gadi-cpu-clx-2095:102038] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x150e56567a5d]
[gadi-cpu-clx-2095:102038] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x150e565243a4]
[gadi-cpu-clx-2095:102038] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x150e568007fb]
[gadi-cpu-clx-2095:102038] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x150e56e2e98e]
[gadi-cpu-clx-2095:102038] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x150e58b892da]
[gadi-cpu-clx-2095:102038] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x150e58e784b2]
[gadi-cpu-clx-2095:102038] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x150e58e99d79]
[gadi-cpu-clx-2095:102038] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x150e5915ba5c]
[gadi-cpu-clx-2095:102038] [13] python3(PyModule_ExecDef+0x78)[0x55578af709d8]
[gadi-cpu-clx-2095:102038] [14] python3(+0x1f2a45)[0x55578af70a45]
[gadi-cpu-clx-2095:102038] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x55578aec75da]
[gadi-cpu-clx-2095:102038] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x55578af4645a]
[gadi-cpu-clx-2095:102038] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x55578ae97505]
[gadi-cpu-clx-2095:102038] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x55578aeb7133]
[gadi-cpu-clx-2095:102038] [19] python3(+0x181455)[0x55578aeff455]
[gadi-cpu-clx-2095:102038] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x55578af45762]
[gadi-cpu-clx-2095:102038] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x55578aeb6d37]
[gadi-cpu-clx-2095:102038] [22] python3(+0x181455)[0x55578aeff455]
[gadi-cpu-clx-2095:102038] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x55578af414d1]
[gadi-cpu-clx-2095:102038] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x55578aeb6d37]
[gadi-cpu-clx-2095:102038] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x55578af412b5]
[gadi-cpu-clx-2095:102038] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x55578aeb6d37]
[gadi-cpu-clx-2095:102038] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x55578af412b5]
[gadi-cpu-clx-2095:102038] [28] python3(_PyObject_FastCallDict+0x1b6)[0x55578ae98ab6]
[gadi-cpu-clx-2095:102038] [29] python3(+0x11b9a2)[0x55578ae999a2]
[gadi-cpu-clx-2095:102038] *** End of error message ***
[gadi-cpu-clx-2095:102041] *** Process received signal ***
[gadi-cpu-clx-2095:102041] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102041] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102041] Failing at address: 0x14712c2b7008
[gadi-cpu-clx-2095:102041] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x14714bcdddd0]
[gadi-cpu-clx-2095:102041] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x147122a550b9]
[gadi-cpu-clx-2095:102041] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x14712284b2a3]
[gadi-cpu-clx-2095:102041] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x147122a54d54]
[gadi-cpu-clx-2095:102041] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x147123b8a61e]
[gadi-cpu-clx-2095:102041] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x147123b41a5d]
[gadi-cpu-clx-2095:102041] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x147123afe3a4]
[gadi-cpu-clx-2095:102041] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x147123dda7fb]
[gadi-cpu-clx-2095:102041] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x14712852798e]
[gadi-cpu-clx-2095:102041] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x14712a2822da]
[gadi-cpu-clx-2095:102041] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x14712a5714b2]
[gadi-cpu-clx-2095:102041] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x14712a592d79]
[gadi-cpu-clx-2095:102041] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x14712a854a5c]
[gadi-cpu-clx-2095:102041] [13] python3(PyModule_ExecDef+0x78)[0x559b512829d8]
[gadi-cpu-clx-2095:102041] [14] python3(+0x1f2a45)[0x559b51282a45]
[gadi-cpu-clx-2095:102041] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x559b511d95da]
[gadi-cpu-clx-2095:102041] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x559b5125845a]
[gadi-cpu-clx-2095:102041] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x559b511a9505]
[gadi-cpu-clx-2095:102041] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x559b511c9133]
[gadi-cpu-clx-2095:102041] [19] python3(+0x181455)[0x559b51211455]
[gadi-cpu-clx-2095:102041] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x559b51257762]
[gadi-cpu-clx-2095:102041] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x559b511c8d37]
[gadi-cpu-clx-2095:102041] [22] python3(+0x181455)[0x559b51211455]
[gadi-cpu-clx-2095:102041] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x559b512534d1]
[gadi-cpu-clx-2095:102041] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x559b511c8d37]
[gadi-cpu-clx-2095:102041] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x559b512532b5]
[gadi-cpu-clx-2095:102041] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x559b511c8d37]
[gadi-cpu-clx-2095:102041] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x559b512532b5]
[gadi-cpu-clx-2095:102041] [28] python3(_PyObject_FastCallDict+0x1b6)[0x559b511aaab6]
[gadi-cpu-clx-2095:102041] [29] python3(+0x11b9a2)[0x559b511ab9a2]
[gadi-cpu-clx-2095:102041] *** End of error message ***
[gadi-cpu-clx-2095:102039] *** Process received signal ***
[gadi-cpu-clx-2095:102039] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102039] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102039] Failing at address: 0x14a67d9b2008
[gadi-cpu-clx-2095:102039] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x14a69d3d8dd0]
[gadi-cpu-clx-2095:102039] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x14a6782590b9]
[gadi-cpu-clx-2095:102039] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x14a67804f2a3]
[gadi-cpu-clx-2095:102039] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x14a678258d54]
[gadi-cpu-clx-2095:102039] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x14a6793a461e]
[gadi-cpu-clx-2095:102039] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x14a67935ba5d]
[gadi-cpu-clx-2095:102039] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x14a6793183a4]
[gadi-cpu-clx-2095:102039] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x14a6795f47fb]
[gadi-cpu-clx-2095:102039] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x14a679c2298e]
[gadi-cpu-clx-2095:102039] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x14a67b97d2da]
[gadi-cpu-clx-2095:102039] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x14a67bc6c4b2]
[gadi-cpu-clx-2095:102039] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x14a67bc8dd79]
[gadi-cpu-clx-2095:102039] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x14a67bf4fa5c]
[gadi-cpu-clx-2095:102039] [13] python3(PyModule_ExecDef+0x78)[0x564d2b8ff9d8]
[gadi-cpu-clx-2095:102039] [14] python3(+0x1f2a45)[0x564d2b8ffa45]
[gadi-cpu-clx-2095:102039] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x564d2b8565da]
[gadi-cpu-clx-2095:102039] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x564d2b8d545a]
[gadi-cpu-clx-2095:102039] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x564d2b826505]
[gadi-cpu-clx-2095:102039] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x564d2b846133]
[gadi-cpu-clx-2095:102039] [19] python3(+0x181455)[0x564d2b88e455]
[gadi-cpu-clx-2095:102039] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x564d2b8d4762]
[gadi-cpu-clx-2095:102039] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x564d2b845d37]
[gadi-cpu-clx-2095:102039] [22] python3(+0x181455)[0x564d2b88e455]
[gadi-cpu-clx-2095:102039] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x564d2b8d04d1]
[gadi-cpu-clx-2095:102039] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x564d2b845d37]
[gadi-cpu-clx-2095:102039] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x564d2b8d02b5]
[gadi-cpu-clx-2095:102039] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x564d2b845d37]
[gadi-cpu-clx-2095:102039] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x564d2b8d02b5]
[gadi-cpu-clx-2095:102039] [28] python3(_PyObject_FastCallDict+0x1b6)[0x564d2b827ab6]
[gadi-cpu-clx-2095:102039] [29] python3(+0x11b9a2)[0x564d2b8289a2]
[gadi-cpu-clx-2095:102039] *** End of error message ***
[gadi-cpu-clx-2095:102042] *** Process received signal ***
[gadi-cpu-clx-2095:102042] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102042] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102042] Failing at address: 0x14ea74aa0008
[gadi-cpu-clx-2095:102042] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x14ea944c6dd0]
[gadi-cpu-clx-2095:102042] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x14ea6b1a90b9]
[gadi-cpu-clx-2095:102042] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x14ea6af9f2a3]
[gadi-cpu-clx-2095:102042] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x14ea6b1a8d54]
[gadi-cpu-clx-2095:102042] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x14ea7049261e]
[gadi-cpu-clx-2095:102042] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x14ea70449a5d]
[gadi-cpu-clx-2095:102042] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x14ea704063a4]
[gadi-cpu-clx-2095:102042] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x14ea706e27fb]
[gadi-cpu-clx-2095:102042] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x14ea70d1098e]
[gadi-cpu-clx-2095:102042] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x14ea72aab2da]
[gadi-cpu-clx-2095:102042] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x14ea72d9a4b2]
[gadi-cpu-clx-2095:102042] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x14ea72dbbd79]
[gadi-cpu-clx-2095:102042] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x14ea7307da5c]
[gadi-cpu-clx-2095:102042] [13] python3(PyModule_ExecDef+0x78)[0x5577f65979d8]
[gadi-cpu-clx-2095:102042] [14] python3(+0x1f2a45)[0x5577f6597a45]
[gadi-cpu-clx-2095:102042] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x5577f64ee5da]
[gadi-cpu-clx-2095:102042] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x5577f656d45a]
[gadi-cpu-clx-2095:102042] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x5577f64be505]
[gadi-cpu-clx-2095:102042] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x5577f64de133]
[gadi-cpu-clx-2095:102042] [19] python3(+0x181455)[0x5577f6526455]
[gadi-cpu-clx-2095:102042] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x5577f656c762]
[gadi-cpu-clx-2095:102042] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x5577f64ddd37]
[gadi-cpu-clx-2095:102042] [22] python3(+0x181455)[0x5577f6526455]
[gadi-cpu-clx-2095:102042] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x5577f65684d1]
[gadi-cpu-clx-2095:102042] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x5577f64ddd37]
[gadi-cpu-clx-2095:102042] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x5577f65682b5]
[gadi-cpu-clx-2095:102042] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x5577f64ddd37]
[gadi-cpu-clx-2095:102042] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x5577f65682b5]
[gadi-cpu-clx-2095:102042] [28] python3(_PyObject_FastCallDict+0x1b6)[0x5577f64bfab6]
[gadi-cpu-clx-2095:102042] [29] python3(+0x11b9a2)[0x5577f64c09a2]
[gadi-cpu-clx-2095:102042] *** End of error message ***
[gadi-cpu-clx-2095:102045] *** Process received signal ***
[gadi-cpu-clx-2095:102045] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102045] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102045] Failing at address: 0x1549406c6008
[gadi-cpu-clx-2095:102045] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x1549600ecdd0]
[gadi-cpu-clx-2095:102045] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x154936e8f0b9]
[gadi-cpu-clx-2095:102045] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x154936c852a3]
[gadi-cpu-clx-2095:102045] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x154936e8ed54]
[gadi-cpu-clx-2095:102045] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x154937dc161e]
[gadi-cpu-clx-2095:102045] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x154937d78a5d]
[gadi-cpu-clx-2095:102045] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x154937d353a4]
[gadi-cpu-clx-2095:102045] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x15493c3087fb]
[gadi-cpu-clx-2095:102045] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x15493c93698e]
[gadi-cpu-clx-2095:102045] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x15493e6d12da]
[gadi-cpu-clx-2095:102045] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x15493e9c04b2]
[gadi-cpu-clx-2095:102045] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x15493e9e1d79]
[gadi-cpu-clx-2095:102045] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x15493eca3a5c]
[gadi-cpu-clx-2095:102045] [13] python3(PyModule_ExecDef+0x78)[0x55bcdea3d9d8]
[gadi-cpu-clx-2095:102045] [14] python3(+0x1f2a45)[0x55bcdea3da45]
[gadi-cpu-clx-2095:102045] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x55bcde9945da]
[gadi-cpu-clx-2095:102045] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x55bcdea1345a]
[gadi-cpu-clx-2095:102045] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x55bcde964505]
[gadi-cpu-clx-2095:102045] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x55bcde984133]
[gadi-cpu-clx-2095:102045] [19] python3(+0x181455)[0x55bcde9cc455]
[gadi-cpu-clx-2095:102045] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x55bcdea12762]
[gadi-cpu-clx-2095:102045] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x55bcde983d37]
[gadi-cpu-clx-2095:102045] [22] python3(+0x181455)[0x55bcde9cc455]
[gadi-cpu-clx-2095:102045] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x55bcdea0e4d1]
[gadi-cpu-clx-2095:102045] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x55bcde983d37]
[gadi-cpu-clx-2095:102045] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x55bcdea0e2b5]
[gadi-cpu-clx-2095:102045] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x55bcde983d37]
[gadi-cpu-clx-2095:102045] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x55bcdea0e2b5]
[gadi-cpu-clx-2095:102045] [28] python3(_PyObject_FastCallDict+0x1b6)[0x55bcde965ab6]
[gadi-cpu-clx-2095:102045] [29] python3(+0x11b9a2)[0x55bcde9669a2]
[gadi-cpu-clx-2095:102045] *** End of error message ***
[gadi-cpu-clx-2095:102043] *** Process received signal ***
[gadi-cpu-clx-2095:102043] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102043] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102043] Failing at address: 0x152b998e6008
[gadi-cpu-clx-2095:102043] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x152bb930cdd0]
[gadi-cpu-clx-2095:102043] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x152b9418d0b9]
[gadi-cpu-clx-2095:102043] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x152b8fdf82a3]
[gadi-cpu-clx-2095:102043] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x152b9418cd54]
[gadi-cpu-clx-2095:102043] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x152b952d861e]
[gadi-cpu-clx-2095:102043] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x152b9528fa5d]
[gadi-cpu-clx-2095:102043] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x152b9524c3a4]
[gadi-cpu-clx-2095:102043] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x152b955287fb]
[gadi-cpu-clx-2095:102043] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x152b95b5698e]
[gadi-cpu-clx-2095:102043] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x152b978b12da]
[gadi-cpu-clx-2095:102043] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x152b97ba04b2]
[gadi-cpu-clx-2095:102043] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x152b97bc1d79]
[gadi-cpu-clx-2095:102043] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x152b97e83a5c]
[gadi-cpu-clx-2095:102043] [13] python3(PyModule_ExecDef+0x78)[0x555c4e7ad9d8]
[gadi-cpu-clx-2095:102043] [14] python3(+0x1f2a45)[0x555c4e7ada45]
[gadi-cpu-clx-2095:102043] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x555c4e7045da]
[gadi-cpu-clx-2095:102043] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x555c4e78345a]
[gadi-cpu-clx-2095:102043] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x555c4e6d4505]
[gadi-cpu-clx-2095:102043] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x555c4e6f4133]
[gadi-cpu-clx-2095:102043] [19] python3(+0x181455)[0x555c4e73c455]
[gadi-cpu-clx-2095:102043] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x555c4e782762]
[gadi-cpu-clx-2095:102043] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x555c4e6f3d37]
[gadi-cpu-clx-2095:102043] [22] python3(+0x181455)[0x555c4e73c455]
[gadi-cpu-clx-2095:102043] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x555c4e77e4d1]
[gadi-cpu-clx-2095:102043] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x555c4e6f3d37]
[gadi-cpu-clx-2095:102043] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x555c4e77e2b5]
[gadi-cpu-clx-2095:102043] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x555c4e6f3d37]
[gadi-cpu-clx-2095:102043] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x555c4e77e2b5]
[gadi-cpu-clx-2095:102043] [28] python3(_PyObject_FastCallDict+0x1b6)[0x555c4e6d5ab6]
[gadi-cpu-clx-2095:102043] [29] python3(+0x11b9a2)[0x555c4e6d69a2]
[gadi-cpu-clx-2095:102043] *** End of error message ***
[gadi-cpu-clx-2095:102048] *** Process received signal ***
[gadi-cpu-clx-2095:102048] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102048] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102048] Failing at address: 0x146ea122c008
[gadi-cpu-clx-2095:102048] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x146ec0c52dd0]
[gadi-cpu-clx-2095:102048] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x146e979e10b9]
[gadi-cpu-clx-2095:102048] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x146e977d72a3]
[gadi-cpu-clx-2095:102048] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x146e979e0d54]
[gadi-cpu-clx-2095:102048] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x146e9cc1e61e]
[gadi-cpu-clx-2095:102048] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x146e9cbd5a5d]
[gadi-cpu-clx-2095:102048] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x146e9cb923a4]
[gadi-cpu-clx-2095:102048] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x146e9ce6e7fb]
[gadi-cpu-clx-2095:102048] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x146e9d49c98e]
[gadi-cpu-clx-2095:102048] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x146e9f2372da]
[gadi-cpu-clx-2095:102048] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x146e9f5264b2]
[gadi-cpu-clx-2095:102048] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x146e9f547d79]
[gadi-cpu-clx-2095:102048] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x146e9f809a5c]
[gadi-cpu-clx-2095:102048] [13] python3(PyModule_ExecDef+0x78)[0x562fff0319d8]
[gadi-cpu-clx-2095:102048] [14] python3(+0x1f2a45)[0x562fff031a45]
[gadi-cpu-clx-2095:102048] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x562ffef885da]
[gadi-cpu-clx-2095:102048] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x562fff00745a]
[gadi-cpu-clx-2095:102048] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x562ffef58505]
[gadi-cpu-clx-2095:102048] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x562ffef78133]
[gadi-cpu-clx-2095:102048] [19] python3(+0x181455)[0x562ffefc0455]
[gadi-cpu-clx-2095:102048] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x562fff006762]
[gadi-cpu-clx-2095:102048] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x562ffef77d37]
[gadi-cpu-clx-2095:102048] [22] python3(+0x181455)[0x562ffefc0455]
[gadi-cpu-clx-2095:102048] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x562fff0024d1]
[gadi-cpu-clx-2095:102048] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x562ffef77d37]
[gadi-cpu-clx-2095:102048] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x562fff0022b5]
[gadi-cpu-clx-2095:102048] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x562ffef77d37]
[gadi-cpu-clx-2095:102048] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x562fff0022b5]
[gadi-cpu-clx-2095:102048] [28] python3(_PyObject_FastCallDict+0x1b6)[0x562ffef59ab6]
[gadi-cpu-clx-2095:102048] [29] python3(+0x11b9a2)[0x562ffef5a9a2]
[gadi-cpu-clx-2095:102048] *** End of error message ***
[gadi-cpu-clx-2095:102046] *** Process received signal ***
[gadi-cpu-clx-2095:102046] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102046] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102046] Failing at address: 0x15259f966008
[gadi-cpu-clx-2095:102046] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x1525bf38cdd0]
[gadi-cpu-clx-2095:102046] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x15259a20d0b9]
[gadi-cpu-clx-2095:102046] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x15259a0032a3]
[gadi-cpu-clx-2095:102046] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x15259a20cd54]
[gadi-cpu-clx-2095:102046] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x15259b35861e]
[gadi-cpu-clx-2095:102046] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x15259b30fa5d]
[gadi-cpu-clx-2095:102046] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x15259b2cc3a4]
[gadi-cpu-clx-2095:102046] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x15259b5a87fb]
[gadi-cpu-clx-2095:102046] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x15259bbd698e]
[gadi-cpu-clx-2095:102046] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x15259d9312da]
[gadi-cpu-clx-2095:102046] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x15259dc204b2]
[gadi-cpu-clx-2095:102046] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x15259dc41d79]
[gadi-cpu-clx-2095:102046] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x15259df03a5c]
[gadi-cpu-clx-2095:102046] [13] python3(PyModule_ExecDef+0x78)[0x558ee0b8e9d8]
[gadi-cpu-clx-2095:102046] [14] python3(+0x1f2a45)[0x558ee0b8ea45]
[gadi-cpu-clx-2095:102046] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x558ee0ae55da]
[gadi-cpu-clx-2095:102046] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x558ee0b6445a]
[gadi-cpu-clx-2095:102046] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x558ee0ab5505]
[gadi-cpu-clx-2095:102046] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x558ee0ad5133]
[gadi-cpu-clx-2095:102046] [19] python3(+0x181455)[0x558ee0b1d455]
[gadi-cpu-clx-2095:102046] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x558ee0b63762]
[gadi-cpu-clx-2095:102046] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x558ee0ad4d37]
[gadi-cpu-clx-2095:102046] [22] python3(+0x181455)[0x558ee0b1d455]
[gadi-cpu-clx-2095:102046] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x558ee0b5f4d1]
[gadi-cpu-clx-2095:102046] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x558ee0ad4d37]
[gadi-cpu-clx-2095:102046] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x558ee0b5f2b5]
[gadi-cpu-clx-2095:102046] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x558ee0ad4d37]
[gadi-cpu-clx-2095:102046] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x558ee0b5f2b5]
[gadi-cpu-clx-2095:102046] [28] python3(_PyObject_FastCallDict+0x1b6)[0x558ee0ab6ab6]
[gadi-cpu-clx-2095:102046] [29] python3(+0x11b9a2)[0x558ee0ab79a2]
[gadi-cpu-clx-2095:102046] *** End of error message ***
[gadi-cpu-clx-2095:102049] *** Process received signal ***
[gadi-cpu-clx-2095:102049] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102049] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102049] Failing at address: 0x1488eee82008
[gadi-cpu-clx-2095:102049] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x14890e8a8dd0]
[gadi-cpu-clx-2095:102049] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x1488e97290b9]
[gadi-cpu-clx-2095:102049] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x1488e951f2a3]
[gadi-cpu-clx-2095:102049] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x1488e9728d54]
[gadi-cpu-clx-2095:102049] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x1488ea87461e]
[gadi-cpu-clx-2095:102049] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x1488ea82ba5d]
[gadi-cpu-clx-2095:102049] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x1488ea7e83a4]
[gadi-cpu-clx-2095:102049] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x1488eaac47fb]
[gadi-cpu-clx-2095:102049] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x1488eb0f298e]
[gadi-cpu-clx-2095:102049] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x1488ece4d2da]
[gadi-cpu-clx-2095:102049] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x1488ed13c4b2]
[gadi-cpu-clx-2095:102049] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x1488ed15dd79]
[gadi-cpu-clx-2095:102049] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x1488ed41fa5c]
[gadi-cpu-clx-2095:102049] [13] python3(PyModule_ExecDef+0x78)[0x564ab36309d8]
[gadi-cpu-clx-2095:102049] [14] python3(+0x1f2a45)[0x564ab3630a45]
[gadi-cpu-clx-2095:102049] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x564ab35875da]
[gadi-cpu-clx-2095:102049] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x564ab360645a]
[gadi-cpu-clx-2095:102049] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x564ab3557505]
[gadi-cpu-clx-2095:102049] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x564ab3577133]
[gadi-cpu-clx-2095:102049] [19] python3(+0x181455)[0x564ab35bf455]
[gadi-cpu-clx-2095:102049] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x564ab3605762]
[gadi-cpu-clx-2095:102049] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x564ab3576d37]
[gadi-cpu-clx-2095:102049] [22] python3(+0x181455)[0x564ab35bf455]
[gadi-cpu-clx-2095:102049] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x564ab36014d1]
[gadi-cpu-clx-2095:102049] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x564ab3576d37]
[gadi-cpu-clx-2095:102049] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x564ab36012b5]
[gadi-cpu-clx-2095:102049] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x564ab3576d37]
[gadi-cpu-clx-2095:102049] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x564ab36012b5]
[gadi-cpu-clx-2095:102049] [28] python3(_PyObject_FastCallDict+0x1b6)[0x564ab3558ab6]
[gadi-cpu-clx-2095:102049] [29] python3(+0x11b9a2)[0x564ab35599a2]
[gadi-cpu-clx-2095:102049] *** End of error message ***
[gadi-cpu-clx-2095:102072] *** Process received signal ***
[gadi-cpu-clx-2095:102072] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102072] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102072] Failing at address: 0x152a3083b008
[gadi-cpu-clx-2095:102072] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x152a50261dd0]
[gadi-cpu-clx-2095:102072] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x152a26f900b9]
[gadi-cpu-clx-2095:102072] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x152a26d862a3]
[gadi-cpu-clx-2095:102072] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x152a26f8fd54]
[gadi-cpu-clx-2095:102072] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x152a2c22d61e]
[gadi-cpu-clx-2095:102072] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x152a2c1e4a5d]
[gadi-cpu-clx-2095:102072] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x152a2c1a13a4]
[gadi-cpu-clx-2095:102072] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x152a2c47d7fb]
[gadi-cpu-clx-2095:102072] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x152a2caab98e]
[gadi-cpu-clx-2095:102072] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x152a2e8062da]
[gadi-cpu-clx-2095:102072] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x152a2eaf54b2]
[gadi-cpu-clx-2095:102072] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x152a2eb16d79]
[gadi-cpu-clx-2095:102072] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x152a2edd8a5c]
[gadi-cpu-clx-2095:102072] [13] python3(PyModule_ExecDef+0x78)[0x5587973e19d8]
[gadi-cpu-clx-2095:102072] [14] python3(+0x1f2a45)[0x5587973e1a45]
[gadi-cpu-clx-2095:102072] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x5587973385da]
[gadi-cpu-clx-2095:102072] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x5587973b745a]
[gadi-cpu-clx-2095:102072] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x558797308505]
[gadi-cpu-clx-2095:102072] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x558797328133]
[gadi-cpu-clx-2095:102072] [19] python3(+0x181455)[0x558797370455]
[gadi-cpu-clx-2095:102072] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x5587973b6762]
[gadi-cpu-clx-2095:102072] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x558797327d37]
[gadi-cpu-clx-2095:102072] [22] python3(+0x181455)[0x558797370455]
[gadi-cpu-clx-2095:102072] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x5587973b24d1]
[gadi-cpu-clx-2095:102072] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x558797327d37]
[gadi-cpu-clx-2095:102072] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x5587973b22b5]
[gadi-cpu-clx-2095:102072] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x558797327d37]
[gadi-cpu-clx-2095:102072] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x5587973b22b5]
[gadi-cpu-clx-2095:102072] [28] python3(_PyObject_FastCallDict+0x1b6)[0x558797309ab6]
[gadi-cpu-clx-2095:102072] [29] python3(+0x11b9a2)[0x55879730a9a2]
[gadi-cpu-clx-2095:102072] *** End of error message ***
[gadi-cpu-clx-2095:102052] *** Process received signal ***
[gadi-cpu-clx-2095:102052] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102052] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102052] Failing at address: 0x14df6f885008
[gadi-cpu-clx-2095:102052] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x14df8f2abdd0]
[gadi-cpu-clx-2095:102052] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x14df6a12c0b9]
[gadi-cpu-clx-2095:102052] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x14df69f222a3]
[gadi-cpu-clx-2095:102052] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x14df6a12bd54]
[gadi-cpu-clx-2095:102052] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x14df6b27761e]
[gadi-cpu-clx-2095:102052] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x14df6b22ea5d]
[gadi-cpu-clx-2095:102052] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x14df6b1eb3a4]
[gadi-cpu-clx-2095:102052] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x14df6b4c77fb]
[gadi-cpu-clx-2095:102052] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x14df6baf598e]
[gadi-cpu-clx-2095:102052] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x14df6d8902da]
[gadi-cpu-clx-2095:102052] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x14df6db7f4b2]
[gadi-cpu-clx-2095:102052] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x14df6dba0d79]
[gadi-cpu-clx-2095:102052] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x14df6de62a5c]
[gadi-cpu-clx-2095:102052] [13] python3(PyModule_ExecDef+0x78)[0x561fc2c509d8]
[gadi-cpu-clx-2095:102052] [14] python3(+0x1f2a45)[0x561fc2c50a45]
[gadi-cpu-clx-2095:102052] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x561fc2ba75da]
[gadi-cpu-clx-2095:102052] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x561fc2c2645a]
[gadi-cpu-clx-2095:102052] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x561fc2b77505]
[gadi-cpu-clx-2095:102052] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x561fc2b97133]
[gadi-cpu-clx-2095:102052] [19] python3(+0x181455)[0x561fc2bdf455]
[gadi-cpu-clx-2095:102052] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x561fc2c25762]
[gadi-cpu-clx-2095:102052] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x561fc2b96d37]
[gadi-cpu-clx-2095:102052] [22] python3(+0x181455)[0x561fc2bdf455]
[gadi-cpu-clx-2095:102052] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x561fc2c214d1]
[gadi-cpu-clx-2095:102052] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x561fc2b96d37]
[gadi-cpu-clx-2095:102052] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x561fc2c212b5]
[gadi-cpu-clx-2095:102052] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x561fc2b96d37]
[gadi-cpu-clx-2095:102052] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x561fc2c212b5]
[gadi-cpu-clx-2095:102052] [28] python3(_PyObject_FastCallDict+0x1b6)[0x561fc2b78ab6]
[gadi-cpu-clx-2095:102052] [29] python3(+0x11b9a2)[0x561fc2b799a2]
[gadi-cpu-clx-2095:102052] *** End of error message ***
[gadi-cpu-clx-2095:102050] *** Process received signal ***
[gadi-cpu-clx-2095:102050] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102050] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102050] Failing at address: 0x154ed5525008
[gadi-cpu-clx-2095:102050] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x154ef4f4bdd0]
[gadi-cpu-clx-2095:102050] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x154ecbbeb0b9]
[gadi-cpu-clx-2095:102050] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x154ecb9e12a3]
[gadi-cpu-clx-2095:102050] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x154ecbbead54]
[gadi-cpu-clx-2095:102050] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x154ed0f1761e]
[gadi-cpu-clx-2095:102050] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x154ed0ecea5d]
[gadi-cpu-clx-2095:102050] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x154ed0e8b3a4]
[gadi-cpu-clx-2095:102050] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x154ed11677fb]
[gadi-cpu-clx-2095:102050] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x154ed179598e]
[gadi-cpu-clx-2095:102050] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x154ed34f02da]
[gadi-cpu-clx-2095:102050] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x154ed37df4b2]
[gadi-cpu-clx-2095:102050] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x154ed3800d79]
[gadi-cpu-clx-2095:102050] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x154ed3ac2a5c]
[gadi-cpu-clx-2095:102050] [13] python3(PyModule_ExecDef+0x78)[0x55f5be5d89d8]
[gadi-cpu-clx-2095:102050] [14] python3(+0x1f2a45)[0x55f5be5d8a45]
[gadi-cpu-clx-2095:102050] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x55f5be52f5da]
[gadi-cpu-clx-2095:102050] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x55f5be5ae45a]
[gadi-cpu-clx-2095:102050] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x55f5be4ff505]
[gadi-cpu-clx-2095:102050] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x55f5be51f133]
[gadi-cpu-clx-2095:102050] [19] python3(+0x181455)[0x55f5be567455]
[gadi-cpu-clx-2095:102050] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x55f5be5ad762]
[gadi-cpu-clx-2095:102050] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x55f5be51ed37]
[gadi-cpu-clx-2095:102050] [22] python3(+0x181455)[0x55f5be567455]
[gadi-cpu-clx-2095:102050] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x55f5be5a94d1]
[gadi-cpu-clx-2095:102050] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x55f5be51ed37]
[gadi-cpu-clx-2095:102050] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x55f5be5a92b5]
[gadi-cpu-clx-2095:102050] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x55f5be51ed37]
[gadi-cpu-clx-2095:102050] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x55f5be5a92b5]
[gadi-cpu-clx-2095:102050] [28] python3(_PyObject_FastCallDict+0x1b6)[0x55f5be500ab6]
[gadi-cpu-clx-2095:102050] [29] python3(+0x11b9a2)[0x55f5be5019a2]
[gadi-cpu-clx-2095:102050] *** End of error message ***
[gadi-cpu-clx-2095:102054] *** Process received signal ***
[gadi-cpu-clx-2095:102054] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102054] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102054] Failing at address: 0x14cff08eb008
[gadi-cpu-clx-2095:102054] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x14d010311dd0]
[gadi-cpu-clx-2095:102054] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x14cfe6f900b9]
[gadi-cpu-clx-2095:102054] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x14cfe6d862a3]
[gadi-cpu-clx-2095:102054] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x14cfe6f8fd54]
[gadi-cpu-clx-2095:102054] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x14cfec2dd61e]
[gadi-cpu-clx-2095:102054] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x14cfec294a5d]
[gadi-cpu-clx-2095:102054] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x14cfec2513a4]
[gadi-cpu-clx-2095:102054] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x14cfec52d7fb]
[gadi-cpu-clx-2095:102054] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x14cfecb5b98e]
[gadi-cpu-clx-2095:102054] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x14cfee8b62da]
[gadi-cpu-clx-2095:102054] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x14cfeeba54b2]
[gadi-cpu-clx-2095:102054] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x14cfeebc6d79]
[gadi-cpu-clx-2095:102054] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x14cfeee88a5c]
[gadi-cpu-clx-2095:102054] [13] python3(PyModule_ExecDef+0x78)[0x562a787c59d8]
[gadi-cpu-clx-2095:102054] [14] python3(+0x1f2a45)[0x562a787c5a45]
[gadi-cpu-clx-2095:102054] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x562a7871c5da]
[gadi-cpu-clx-2095:102054] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x562a7879b45a]
[gadi-cpu-clx-2095:102054] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x562a786ec505]
[gadi-cpu-clx-2095:102054] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x562a7870c133]
[gadi-cpu-clx-2095:102054] [19] python3(+0x181455)[0x562a78754455]
[gadi-cpu-clx-2095:102054] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x562a7879a762]
[gadi-cpu-clx-2095:102054] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x562a7870bd37]
[gadi-cpu-clx-2095:102054] [22] python3(+0x181455)[0x562a78754455]
[gadi-cpu-clx-2095:102054] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x562a787964d1]
[gadi-cpu-clx-2095:102054] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x562a7870bd37]
[gadi-cpu-clx-2095:102054] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x562a787962b5]
[gadi-cpu-clx-2095:102054] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x562a7870bd37]
[gadi-cpu-clx-2095:102054] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x562a787962b5]
[gadi-cpu-clx-2095:102054] [28] python3(_PyObject_FastCallDict+0x1b6)[0x562a786edab6]
[gadi-cpu-clx-2095:102054] [29] python3(+0x11b9a2)[0x562a786ee9a2]
[gadi-cpu-clx-2095:102054] *** End of error message ***
[gadi-cpu-clx-2095:102051] *** Process received signal ***
[gadi-cpu-clx-2095:102051] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102051] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102051] Failing at address: 0x149a11c72008
[gadi-cpu-clx-2095:102051] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x149a31698dd0]
[gadi-cpu-clx-2095:102051] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x149a0c5190b9]
[gadi-cpu-clx-2095:102051] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x149a0c30f2a3]
[gadi-cpu-clx-2095:102051] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x149a0c518d54]
[gadi-cpu-clx-2095:102051] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x149a0d66461e]
[gadi-cpu-clx-2095:102051] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x149a0d61ba5d]
[gadi-cpu-clx-2095:102051] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x149a0d5d83a4]
[gadi-cpu-clx-2095:102051] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x149a0d8b47fb]
[gadi-cpu-clx-2095:102051] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x149a0dee298e]
[gadi-cpu-clx-2095:102051] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x149a0fc3d2da]
[gadi-cpu-clx-2095:102051] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x149a0ff2c4b2]
[gadi-cpu-clx-2095:102051] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x149a0ff4dd79]
[gadi-cpu-clx-2095:102051] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x149a1020fa5c]
[gadi-cpu-clx-2095:102051] [13] python3(PyModule_ExecDef+0x78)[0x564cb5fce9d8]
[gadi-cpu-clx-2095:102051] [14] python3(+0x1f2a45)[0x564cb5fcea45]
[gadi-cpu-clx-2095:102051] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x564cb5f255da]
[gadi-cpu-clx-2095:102051] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x564cb5fa445a]
[gadi-cpu-clx-2095:102051] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x564cb5ef5505]
[gadi-cpu-clx-2095:102051] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x564cb5f15133]
[gadi-cpu-clx-2095:102051] [19] python3(+0x181455)[0x564cb5f5d455]
[gadi-cpu-clx-2095:102051] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x564cb5fa3762]
[gadi-cpu-clx-2095:102051] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x564cb5f14d37]
[gadi-cpu-clx-2095:102051] [22] python3(+0x181455)[0x564cb5f5d455]
[gadi-cpu-clx-2095:102051] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x564cb5f9f4d1]
[gadi-cpu-clx-2095:102051] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x564cb5f14d37]
[gadi-cpu-clx-2095:102051] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x564cb5f9f2b5]
[gadi-cpu-clx-2095:102051] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x564cb5f14d37]
[gadi-cpu-clx-2095:102051] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x564cb5f9f2b5]
[gadi-cpu-clx-2095:102051] [28] python3(_PyObject_FastCallDict+0x1b6)[0x564cb5ef6ab6]
[gadi-cpu-clx-2095:102051] [29] python3(+0x11b9a2)[0x564cb5ef79a2]
[gadi-cpu-clx-2095:102051] *** End of error message ***
[gadi-cpu-clx-2095:102056] *** Process received signal ***
[gadi-cpu-clx-2095:102056] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102056] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102056] Failing at address: 0x150950e2e008
[gadi-cpu-clx-2095:102056] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x150970854dd0]
[gadi-cpu-clx-2095:102056] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x1509475c40b9]
[gadi-cpu-clx-2095:102056] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x1509473ba2a3]
[gadi-cpu-clx-2095:102056] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x1509475c3d54]
[gadi-cpu-clx-2095:102056] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x15094c82061e]
[gadi-cpu-clx-2095:102056] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x15094c7d7a5d]
[gadi-cpu-clx-2095:102056] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x15094c7943a4]
[gadi-cpu-clx-2095:102056] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x15094ca707fb]
[gadi-cpu-clx-2095:102056] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x15094d09e98e]
[gadi-cpu-clx-2095:102056] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x15094edf92da]
[gadi-cpu-clx-2095:102056] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x15094f0e84b2]
[gadi-cpu-clx-2095:102056] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x15094f109d79]
[gadi-cpu-clx-2095:102056] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x15094f3cba5c]
[gadi-cpu-clx-2095:102056] [13] python3(PyModule_ExecDef+0x78)[0x558dd39f89d8]
[gadi-cpu-clx-2095:102056] [14] python3(+0x1f2a45)[0x558dd39f8a45]
[gadi-cpu-clx-2095:102056] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x558dd394f5da]
[gadi-cpu-clx-2095:102056] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x558dd39ce45a]
[gadi-cpu-clx-2095:102056] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x558dd391f505]
[gadi-cpu-clx-2095:102056] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x558dd393f133]
[gadi-cpu-clx-2095:102056] [19] python3(+0x181455)[0x558dd3987455]
[gadi-cpu-clx-2095:102056] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x558dd39cd762]
[gadi-cpu-clx-2095:102056] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x558dd393ed37]
[gadi-cpu-clx-2095:102056] [22] python3(+0x181455)[0x558dd3987455]
[gadi-cpu-clx-2095:102056] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x558dd39c94d1]
[gadi-cpu-clx-2095:102056] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x558dd393ed37]
[gadi-cpu-clx-2095:102056] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x558dd39c92b5]
[gadi-cpu-clx-2095:102056] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x558dd393ed37]
[gadi-cpu-clx-2095:102056] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x558dd39c92b5]
[gadi-cpu-clx-2095:102056] [28] python3(_PyObject_FastCallDict+0x1b6)[0x558dd3920ab6]
[gadi-cpu-clx-2095:102056] [29] python3(+0x11b9a2)[0x558dd39219a2]
[gadi-cpu-clx-2095:102056] *** End of error message ***
[gadi-cpu-clx-2095:102053] *** Process received signal ***
[gadi-cpu-clx-2095:102053] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102053] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102053] Failing at address: 0x151550b34008
[gadi-cpu-clx-2095:102053] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x15157055add0]
[gadi-cpu-clx-2095:102053] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x1515473b20b9]
[gadi-cpu-clx-2095:102053] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x1515471a82a3]
[gadi-cpu-clx-2095:102053] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x1515473b1d54]
[gadi-cpu-clx-2095:102053] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x15154c52661e]
[gadi-cpu-clx-2095:102053] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x15154c4dda5d]
[gadi-cpu-clx-2095:102053] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x15154c49a3a4]
[gadi-cpu-clx-2095:102053] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x15154c7767fb]
[gadi-cpu-clx-2095:102053] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x15154cda498e]
[gadi-cpu-clx-2095:102053] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x15154eaff2da]
[gadi-cpu-clx-2095:102053] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x15154edee4b2]
[gadi-cpu-clx-2095:102053] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x15154ee0fd79]
[gadi-cpu-clx-2095:102053] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x15154f0d1a5c]
[gadi-cpu-clx-2095:102053] [13] python3(PyModule_ExecDef+0x78)[0x5576c6cab9d8]
[gadi-cpu-clx-2095:102053] [14] python3(+0x1f2a45)[0x5576c6caba45]
[gadi-cpu-clx-2095:102053] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x5576c6c025da]
[gadi-cpu-clx-2095:102053] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x5576c6c8145a]
[gadi-cpu-clx-2095:102053] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x5576c6bd2505]
[gadi-cpu-clx-2095:102053] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x5576c6bf2133]
[gadi-cpu-clx-2095:102053] [19] python3(+0x181455)[0x5576c6c3a455]
[gadi-cpu-clx-2095:102053] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x5576c6c80762]
[gadi-cpu-clx-2095:102053] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x5576c6bf1d37]
[gadi-cpu-clx-2095:102053] [22] python3(+0x181455)[0x5576c6c3a455]
[gadi-cpu-clx-2095:102053] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x5576c6c7c4d1]
[gadi-cpu-clx-2095:102053] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x5576c6bf1d37]
[gadi-cpu-clx-2095:102053] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x5576c6c7c2b5]
[gadi-cpu-clx-2095:102053] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x5576c6bf1d37]
[gadi-cpu-clx-2095:102053] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x5576c6c7c2b5]
[gadi-cpu-clx-2095:102053] [28] python3(_PyObject_FastCallDict+0x1b6)[0x5576c6bd3ab6]
[gadi-cpu-clx-2095:102053] [29] python3(+0x11b9a2)[0x5576c6bd49a2]
[gadi-cpu-clx-2095:102053] *** End of error message ***
[gadi-cpu-clx-2095:102057] *** Process received signal ***
[gadi-cpu-clx-2095:102057] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102057] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102057] Failing at address: 0x1467ffa4a008
[gadi-cpu-clx-2095:102057] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x14681f470dd0]
[gadi-cpu-clx-2095:102057] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x1467fa2f10b9]
[gadi-cpu-clx-2095:102057] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x1467fa0e72a3]
[gadi-cpu-clx-2095:102057] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x1467fa2f0d54]
[gadi-cpu-clx-2095:102057] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x1467fb43c61e]
[gadi-cpu-clx-2095:102057] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x1467fb3f3a5d]
[gadi-cpu-clx-2095:102057] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x1467fb3b03a4]
[gadi-cpu-clx-2095:102057] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x1467fb68c7fb]
[gadi-cpu-clx-2095:102057] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x1467fbcba98e]
[gadi-cpu-clx-2095:102057] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x1467fda152da]
[gadi-cpu-clx-2095:102057] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x1467fdd044b2]
[gadi-cpu-clx-2095:102057] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x1467fdd25d79]
[gadi-cpu-clx-2095:102057] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x1467fdfe7a5c]
[gadi-cpu-clx-2095:102057] [13] python3(PyModule_ExecDef+0x78)[0x556d5eb349d8]
[gadi-cpu-clx-2095:102057] [14] python3(+0x1f2a45)[0x556d5eb34a45]
[gadi-cpu-clx-2095:102057] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x556d5ea8b5da]
[gadi-cpu-clx-2095:102057] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x556d5eb0a45a]
[gadi-cpu-clx-2095:102057] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x556d5ea5b505]
[gadi-cpu-clx-2095:102057] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x556d5ea7b133]
[gadi-cpu-clx-2095:102057] [19] python3(+0x181455)[0x556d5eac3455]
[gadi-cpu-clx-2095:102057] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x556d5eb09762]
[gadi-cpu-clx-2095:102057] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x556d5ea7ad37]
[gadi-cpu-clx-2095:102057] [22] python3(+0x181455)[0x556d5eac3455]
[gadi-cpu-clx-2095:102057] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x556d5eb054d1]
[gadi-cpu-clx-2095:102057] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x556d5ea7ad37]
[gadi-cpu-clx-2095:102057] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x556d5eb052b5]
[gadi-cpu-clx-2095:102057] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x556d5ea7ad37]
[gadi-cpu-clx-2095:102057] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x556d5eb052b5]
[gadi-cpu-clx-2095:102057] [28] python3(_PyObject_FastCallDict+0x1b6)[0x556d5ea5cab6]
[gadi-cpu-clx-2095:102057] [29] python3(+0x11b9a2)[0x556d5ea5d9a2]
[gadi-cpu-clx-2095:102057] *** End of error message ***
[gadi-cpu-clx-2095:102077] *** Process received signal ***
[gadi-cpu-clx-2095:102077] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102077] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102077] Failing at address: 0x14ef4bccc008
[gadi-cpu-clx-2095:102077] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x14ef6b6f2dd0]
[gadi-cpu-clx-2095:102077] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x14ef465730b9]
[gadi-cpu-clx-2095:102077] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x14ef463692a3]
[gadi-cpu-clx-2095:102077] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x14ef46572d54]
[gadi-cpu-clx-2095:102077] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x14ef476be61e]
[gadi-cpu-clx-2095:102077] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x14ef47675a5d]
[gadi-cpu-clx-2095:102077] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x14ef476323a4]
[gadi-cpu-clx-2095:102077] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x14ef4790e7fb]
[gadi-cpu-clx-2095:102077] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x14ef47f3c98e]
[gadi-cpu-clx-2095:102077] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x14ef49c972da]
[gadi-cpu-clx-2095:102077] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x14ef49f864b2]
[gadi-cpu-clx-2095:102077] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x14ef49fa7d79]
[gadi-cpu-clx-2095:102077] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x14ef4a269a5c]
[gadi-cpu-clx-2095:102077] [13] python3(PyModule_ExecDef+0x78)[0x55d755afb9d8]
[gadi-cpu-clx-2095:102077] [14] python3(+0x1f2a45)[0x55d755afba45]
[gadi-cpu-clx-2095:102077] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x55d755a525da]
[gadi-cpu-clx-2095:102077] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x55d755ad145a]
[gadi-cpu-clx-2095:102077] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x55d755a22505]
[gadi-cpu-clx-2095:102077] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x55d755a42133]
[gadi-cpu-clx-2095:102077] [19] python3(+0x181455)[0x55d755a8a455]
[gadi-cpu-clx-2095:102077] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x55d755ad0762]
[gadi-cpu-clx-2095:102077] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x55d755a41d37]
[gadi-cpu-clx-2095:102077] [22] python3(+0x181455)[0x55d755a8a455]
[gadi-cpu-clx-2095:102077] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x55d755acc4d1]
[gadi-cpu-clx-2095:102077] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x55d755a41d37]
[gadi-cpu-clx-2095:102077] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x55d755acc2b5]
[gadi-cpu-clx-2095:102077] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x55d755a41d37]
[gadi-cpu-clx-2095:102077] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x55d755acc2b5]
[gadi-cpu-clx-2095:102077] [28] python3(_PyObject_FastCallDict+0x1b6)[0x55d755a23ab6]
[gadi-cpu-clx-2095:102077] [29] python3(+0x11b9a2)[0x55d755a249a2]
[gadi-cpu-clx-2095:102077] *** End of error message ***
[gadi-cpu-clx-2095:102060] *** Process received signal ***
[gadi-cpu-clx-2095:102060] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102060] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102060] Failing at address: 0x14d36d833008
[gadi-cpu-clx-2095:102060] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x14d38d259dd0]
[gadi-cpu-clx-2095:102060] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x14d3680da0b9]
[gadi-cpu-clx-2095:102060] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x14d363df82a3]
[gadi-cpu-clx-2095:102060] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x14d3680d9d54]
[gadi-cpu-clx-2095:102060] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x14d36922561e]
[gadi-cpu-clx-2095:102060] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x14d3691dca5d]
[gadi-cpu-clx-2095:102060] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x14d3691993a4]
[gadi-cpu-clx-2095:102060] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x14d3694757fb]
[gadi-cpu-clx-2095:102060] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x14d369aa398e]
[gadi-cpu-clx-2095:102060] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x14d36b7fe2da]
[gadi-cpu-clx-2095:102060] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x14d36baed4b2]
[gadi-cpu-clx-2095:102060] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x14d36bb0ed79]
[gadi-cpu-clx-2095:102060] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x14d36bdd0a5c]
[gadi-cpu-clx-2095:102060] [13] python3(PyModule_ExecDef+0x78)[0x561d4a1f09d8]
[gadi-cpu-clx-2095:102060] [14] python3(+0x1f2a45)[0x561d4a1f0a45]
[gadi-cpu-clx-2095:102060] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x561d4a1475da]
[gadi-cpu-clx-2095:102060] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x561d4a1c645a]
[gadi-cpu-clx-2095:102060] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x561d4a117505]
[gadi-cpu-clx-2095:102060] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x561d4a137133]
[gadi-cpu-clx-2095:102060] [19] python3(+0x181455)[0x561d4a17f455]
[gadi-cpu-clx-2095:102060] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x561d4a1c5762]
[gadi-cpu-clx-2095:102060] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x561d4a136d37]
[gadi-cpu-clx-2095:102060] [22] python3(+0x181455)[0x561d4a17f455]
[gadi-cpu-clx-2095:102060] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x561d4a1c14d1]
[gadi-cpu-clx-2095:102060] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x561d4a136d37]
[gadi-cpu-clx-2095:102060] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x561d4a1c12b5]
[gadi-cpu-clx-2095:102060] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x561d4a136d37]
[gadi-cpu-clx-2095:102060] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x561d4a1c12b5]
[gadi-cpu-clx-2095:102060] [28] python3(_PyObject_FastCallDict+0x1b6)[0x561d4a118ab6]
[gadi-cpu-clx-2095:102060] [29] python3(+0x11b9a2)[0x561d4a1199a2]
[gadi-cpu-clx-2095:102060] *** End of error message ***
[gadi-cpu-clx-2095:102059] *** Process received signal ***
[gadi-cpu-clx-2095:102059] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102059] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102059] Failing at address: 0x14a7e2b61008
[gadi-cpu-clx-2095:102059] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x14a802587dd0]
[gadi-cpu-clx-2095:102059] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x14a7dd4080b9]
[gadi-cpu-clx-2095:102059] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x14a7dd1fe2a3]
[gadi-cpu-clx-2095:102059] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x14a7dd407d54]
[gadi-cpu-clx-2095:102059] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x14a7de55361e]
[gadi-cpu-clx-2095:102059] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x14a7de50aa5d]
[gadi-cpu-clx-2095:102059] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x14a7de4c73a4]
[gadi-cpu-clx-2095:102059] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x14a7de7a37fb]
[gadi-cpu-clx-2095:102059] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x14a7dedd198e]
[gadi-cpu-clx-2095:102059] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x14a7e0b2c2da]
[gadi-cpu-clx-2095:102059] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x14a7e0e1b4b2]
[gadi-cpu-clx-2095:102059] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x14a7e0e3cd79]
[gadi-cpu-clx-2095:102059] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x14a7e10fea5c]
[gadi-cpu-clx-2095:102059] [13] python3(PyModule_ExecDef+0x78)[0x55b71639d9d8]
[gadi-cpu-clx-2095:102059] [14] python3(+0x1f2a45)[0x55b71639da45]
[gadi-cpu-clx-2095:102059] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x55b7162f45da]
[gadi-cpu-clx-2095:102059] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x55b71637345a]
[gadi-cpu-clx-2095:102059] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x55b7162c4505]
[gadi-cpu-clx-2095:102059] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x55b7162e4133]
[gadi-cpu-clx-2095:102059] [19] python3(+0x181455)[0x55b71632c455]
[gadi-cpu-clx-2095:102059] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x55b716372762]
[gadi-cpu-clx-2095:102059] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x55b7162e3d37]
[gadi-cpu-clx-2095:102059] [22] python3(+0x181455)[0x55b71632c455]
[gadi-cpu-clx-2095:102059] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x55b71636e4d1]
[gadi-cpu-clx-2095:102059] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x55b7162e3d37]
[gadi-cpu-clx-2095:102059] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x55b71636e2b5]
[gadi-cpu-clx-2095:102059] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x55b7162e3d37]
[gadi-cpu-clx-2095:102059] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x55b71636e2b5]
[gadi-cpu-clx-2095:102059] [28] python3(_PyObject_FastCallDict+0x1b6)[0x55b7162c5ab6]
[gadi-cpu-clx-2095:102059] [29] python3(+0x11b9a2)[0x55b7162c69a2]
[gadi-cpu-clx-2095:102059] *** End of error message ***
[gadi-cpu-clx-2095:102061] *** Process received signal ***
[gadi-cpu-clx-2095:102061] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102061] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102061] Failing at address: 0x14abda718008
[gadi-cpu-clx-2095:102061] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x14abfa13edd0]
[gadi-cpu-clx-2095:102061] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x14abd4fbf0b9]
[gadi-cpu-clx-2095:102061] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x14abd4db52a3]
[gadi-cpu-clx-2095:102061] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x14abd4fbed54]
[gadi-cpu-clx-2095:102061] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x14abd610a61e]
[gadi-cpu-clx-2095:102061] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x14abd60c1a5d]
[gadi-cpu-clx-2095:102061] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x14abd607e3a4]
[gadi-cpu-clx-2095:102061] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x14abd635a7fb]
[gadi-cpu-clx-2095:102061] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x14abd698898e]
[gadi-cpu-clx-2095:102061] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x14abd86e32da]
[gadi-cpu-clx-2095:102061] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x14abd89d24b2]
[gadi-cpu-clx-2095:102061] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x14abd89f3d79]
[gadi-cpu-clx-2095:102061] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x14abd8cb5a5c]
[gadi-cpu-clx-2095:102061] [13] python3(PyModule_ExecDef+0x78)[0x55b1c4f929d8]
[gadi-cpu-clx-2095:102061] [14] python3(+0x1f2a45)[0x55b1c4f92a45]
[gadi-cpu-clx-2095:102061] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x55b1c4ee95da]
[gadi-cpu-clx-2095:102061] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x55b1c4f6845a]
[gadi-cpu-clx-2095:102061] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x55b1c4eb9505]
[gadi-cpu-clx-2095:102061] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x55b1c4ed9133]
[gadi-cpu-clx-2095:102061] [19] python3(+0x181455)[0x55b1c4f21455]
[gadi-cpu-clx-2095:102061] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x55b1c4f67762]
[gadi-cpu-clx-2095:102061] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x55b1c4ed8d37]
[gadi-cpu-clx-2095:102061] [22] python3(+0x181455)[0x55b1c4f21455]
[gadi-cpu-clx-2095:102061] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x55b1c4f634d1]
[gadi-cpu-clx-2095:102061] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x55b1c4ed8d37]
[gadi-cpu-clx-2095:102061] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x55b1c4f632b5]
[gadi-cpu-clx-2095:102061] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x55b1c4ed8d37]
[gadi-cpu-clx-2095:102061] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x55b1c4f632b5]
[gadi-cpu-clx-2095:102061] [28] python3(_PyObject_FastCallDict+0x1b6)[0x55b1c4ebaab6]
[gadi-cpu-clx-2095:102061] [29] python3(+0x11b9a2)[0x55b1c4ebb9a2]
[gadi-cpu-clx-2095:102061] *** End of error message ***
[gadi-cpu-clx-2095:102058] *** Process received signal ***
[gadi-cpu-clx-2095:102058] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102058] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102058] Failing at address: 0x1546a425a008
[gadi-cpu-clx-2095:102058] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x1546c3c80dd0]
[gadi-cpu-clx-2095:102058] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x15469aa550b9]
[gadi-cpu-clx-2095:102058] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x15469a84b2a3]
[gadi-cpu-clx-2095:102058] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x15469aa54d54]
[gadi-cpu-clx-2095:102058] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x15469bb8a61e]
[gadi-cpu-clx-2095:102058] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x15469bb41a5d]
[gadi-cpu-clx-2095:102058] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x15469bafe3a4]
[gadi-cpu-clx-2095:102058] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x15469bdda7fb]
[gadi-cpu-clx-2095:102058] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x1546a04ca98e]
[gadi-cpu-clx-2095:102058] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x1546a22252da]
[gadi-cpu-clx-2095:102058] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x1546a25144b2]
[gadi-cpu-clx-2095:102058] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x1546a2535d79]
[gadi-cpu-clx-2095:102058] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x1546a27f7a5c]
[gadi-cpu-clx-2095:102058] [13] python3(PyModule_ExecDef+0x78)[0x564cb847e9d8]
[gadi-cpu-clx-2095:102058] [14] python3(+0x1f2a45)[0x564cb847ea45]
[gadi-cpu-clx-2095:102058] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x564cb83d55da]
[gadi-cpu-clx-2095:102058] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x564cb845445a]
[gadi-cpu-clx-2095:102058] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x564cb83a5505]
[gadi-cpu-clx-2095:102058] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x564cb83c5133]
[gadi-cpu-clx-2095:102058] [19] python3(+0x181455)[0x564cb840d455]
[gadi-cpu-clx-2095:102058] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x564cb8453762]
[gadi-cpu-clx-2095:102058] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x564cb83c4d37]
[gadi-cpu-clx-2095:102058] [22] python3(+0x181455)[0x564cb840d455]
[gadi-cpu-clx-2095:102058] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x564cb844f4d1]
[gadi-cpu-clx-2095:102058] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x564cb83c4d37]
[gadi-cpu-clx-2095:102058] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x564cb844f2b5]
[gadi-cpu-clx-2095:102058] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x564cb83c4d37]
[gadi-cpu-clx-2095:102058] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x564cb844f2b5]
[gadi-cpu-clx-2095:102058] [28] python3(_PyObject_FastCallDict+0x1b6)[0x564cb83a6ab6]
[gadi-cpu-clx-2095:102058] [29] python3(+0x11b9a2)[0x564cb83a79a2]
[gadi-cpu-clx-2095:102058] *** End of error message ***
[gadi-cpu-clx-2095:102063] *** Process received signal ***
[gadi-cpu-clx-2095:102063] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102063] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102063] Failing at address: 0x14bad163d008
[gadi-cpu-clx-2095:102063] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x14baf1063dd0]
[gadi-cpu-clx-2095:102063] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x14bac7dfd0b9]
[gadi-cpu-clx-2095:102063] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x14bac7bf32a3]
[gadi-cpu-clx-2095:102063] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x14bac7dfcd54]
[gadi-cpu-clx-2095:102063] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x14bacd02f61e]
[gadi-cpu-clx-2095:102063] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x14baccfe6a5d]
[gadi-cpu-clx-2095:102063] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x14baccfa33a4]
[gadi-cpu-clx-2095:102063] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x14bacd27f7fb]
[gadi-cpu-clx-2095:102063] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x14bacd8ad98e]
[gadi-cpu-clx-2095:102063] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x14bacf6482da]
[gadi-cpu-clx-2095:102063] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x14bacf9374b2]
[gadi-cpu-clx-2095:102063] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x14bacf958d79]
[gadi-cpu-clx-2095:102063] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x14bacfc1aa5c]
[gadi-cpu-clx-2095:102063] [13] python3(PyModule_ExecDef+0x78)[0x5589512829d8]
[gadi-cpu-clx-2095:102063] [14] python3(+0x1f2a45)[0x558951282a45]
[gadi-cpu-clx-2095:102063] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x5589511d95da]
[gadi-cpu-clx-2095:102063] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x55895125845a]
[gadi-cpu-clx-2095:102063] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x5589511a9505]
[gadi-cpu-clx-2095:102063] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x5589511c9133]
[gadi-cpu-clx-2095:102063] [19] python3(+0x181455)[0x558951211455]
[gadi-cpu-clx-2095:102063] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x558951257762]
[gadi-cpu-clx-2095:102063] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x5589511c8d37]
[gadi-cpu-clx-2095:102063] [22] python3(+0x181455)[0x558951211455]
[gadi-cpu-clx-2095:102063] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x5589512534d1]
[gadi-cpu-clx-2095:102063] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x5589511c8d37]
[gadi-cpu-clx-2095:102063] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x5589512532b5]
[gadi-cpu-clx-2095:102063] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x5589511c8d37]
[gadi-cpu-clx-2095:102063] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x5589512532b5]
[gadi-cpu-clx-2095:102063] [28] python3(_PyObject_FastCallDict+0x1b6)[0x5589511aaab6]
[gadi-cpu-clx-2095:102063] [29] python3(+0x11b9a2)[0x5589511ab9a2]
[gadi-cpu-clx-2095:102063] *** End of error message ***
[gadi-cpu-clx-2095:102064] *** Process received signal ***
[gadi-cpu-clx-2095:102064] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102064] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102064] Failing at address: 0x151298d1f008
[gadi-cpu-clx-2095:102064] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x1512b8745dd0]
[gadi-cpu-clx-2095:102064] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x15128f5c40b9]
[gadi-cpu-clx-2095:102064] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x15128f3ba2a3]
[gadi-cpu-clx-2095:102064] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x15128f5c3d54]
[gadi-cpu-clx-2095:102064] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x15129471161e]
[gadi-cpu-clx-2095:102064] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x1512946c8a5d]
[gadi-cpu-clx-2095:102064] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x1512946853a4]
[gadi-cpu-clx-2095:102064] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x1512949617fb]
[gadi-cpu-clx-2095:102064] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x151294f8f98e]
[gadi-cpu-clx-2095:102064] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x151296cea2da]
[gadi-cpu-clx-2095:102064] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x151296fd94b2]
[gadi-cpu-clx-2095:102064] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x151296ffad79]
[gadi-cpu-clx-2095:102064] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x1512972bca5c]
[gadi-cpu-clx-2095:102064] [13] python3(PyModule_ExecDef+0x78)[0x55f9aa6c29d8]
[gadi-cpu-clx-2095:102064] [14] python3(+0x1f2a45)[0x55f9aa6c2a45]
[gadi-cpu-clx-2095:102064] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x55f9aa6195da]
[gadi-cpu-clx-2095:102064] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x55f9aa69845a]
[gadi-cpu-clx-2095:102064] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x55f9aa5e9505]
[gadi-cpu-clx-2095:102064] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x55f9aa609133]
[gadi-cpu-clx-2095:102064] [19] python3(+0x181455)[0x55f9aa651455]
[gadi-cpu-clx-2095:102064] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x55f9aa697762]
[gadi-cpu-clx-2095:102064] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x55f9aa608d37]
[gadi-cpu-clx-2095:102064] [22] python3(+0x181455)[0x55f9aa651455]
[gadi-cpu-clx-2095:102064] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x55f9aa6934d1]
[gadi-cpu-clx-2095:102064] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x55f9aa608d37]
[gadi-cpu-clx-2095:102064] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x55f9aa6932b5]
[gadi-cpu-clx-2095:102064] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x55f9aa608d37]
[gadi-cpu-clx-2095:102064] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x55f9aa6932b5]
[gadi-cpu-clx-2095:102064] [28] python3(_PyObject_FastCallDict+0x1b6)[0x55f9aa5eaab6]
[gadi-cpu-clx-2095:102064] [29] python3(+0x11b9a2)[0x55f9aa5eb9a2]
[gadi-cpu-clx-2095:102064] *** End of error message ***
[gadi-cpu-clx-2095:102065] *** Process received signal ***
[gadi-cpu-clx-2095:102065] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102065] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102065] Failing at address: 0x15414c6cb008
[gadi-cpu-clx-2095:102065] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x15416c0f1dd0]
[gadi-cpu-clx-2095:102065] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x154142e8f0b9]
[gadi-cpu-clx-2095:102065] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x154142c852a3]
[gadi-cpu-clx-2095:102065] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x154142e8ed54]
[gadi-cpu-clx-2095:102065] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x154143dc161e]
[gadi-cpu-clx-2095:102065] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x154143d78a5d]
[gadi-cpu-clx-2095:102065] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x154143d353a4]
[gadi-cpu-clx-2095:102065] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x15414830d7fb]
[gadi-cpu-clx-2095:102065] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x15414893b98e]
[gadi-cpu-clx-2095:102065] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x15414a6962da]
[gadi-cpu-clx-2095:102065] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x15414a9854b2]
[gadi-cpu-clx-2095:102065] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x15414a9a6d79]
[gadi-cpu-clx-2095:102065] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x15414ac68a5c]
[gadi-cpu-clx-2095:102065] [13] python3(PyModule_ExecDef+0x78)[0x55e9499899d8]
[gadi-cpu-clx-2095:102065] [14] python3(+0x1f2a45)[0x55e949989a45]
[gadi-cpu-clx-2095:102065] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x55e9498e05da]
[gadi-cpu-clx-2095:102065] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x55e94995f45a]
[gadi-cpu-clx-2095:102065] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x55e9498b0505]
[gadi-cpu-clx-2095:102065] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x55e9498d0133]
[gadi-cpu-clx-2095:102065] [19] python3(+0x181455)[0x55e949918455]
[gadi-cpu-clx-2095:102065] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x55e94995e762]
[gadi-cpu-clx-2095:102065] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x55e9498cfd37]
[gadi-cpu-clx-2095:102065] [22] python3(+0x181455)[0x55e949918455]
[gadi-cpu-clx-2095:102065] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x55e94995a4d1]
[gadi-cpu-clx-2095:102065] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x55e9498cfd37]
[gadi-cpu-clx-2095:102065] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x55e94995a2b5]
[gadi-cpu-clx-2095:102065] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x55e9498cfd37]
[gadi-cpu-clx-2095:102065] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x55e94995a2b5]
[gadi-cpu-clx-2095:102065] [28] python3(_PyObject_FastCallDict+0x1b6)[0x55e9498b1ab6]
[gadi-cpu-clx-2095:102065] [29] python3(+0x11b9a2)[0x55e9498b29a2]
[gadi-cpu-clx-2095:102065] *** End of error message ***
[gadi-cpu-clx-2095:102062] *** Process received signal ***
[gadi-cpu-clx-2095:102062] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102062] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102062] Failing at address: 0x1484ae6bf008
[gadi-cpu-clx-2095:102062] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x1484ce0e5dd0]
[gadi-cpu-clx-2095:102062] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x1484a8f660b9]
[gadi-cpu-clx-2095:102062] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x1484a8d5c2a3]
[gadi-cpu-clx-2095:102062] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x1484a8f65d54]
[gadi-cpu-clx-2095:102062] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x1484aa0b161e]
[gadi-cpu-clx-2095:102062] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x1484aa068a5d]
[gadi-cpu-clx-2095:102062] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x1484aa0253a4]
[gadi-cpu-clx-2095:102062] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x1484aa3017fb]
[gadi-cpu-clx-2095:102062] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x1484aa92f98e]
[gadi-cpu-clx-2095:102062] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x1484ac68a2da]
[gadi-cpu-clx-2095:102062] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x1484ac9794b2]
[gadi-cpu-clx-2095:102062] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x1484ac99ad79]
[gadi-cpu-clx-2095:102062] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x1484acc5ca5c]
[gadi-cpu-clx-2095:102062] [13] python3(PyModule_ExecDef+0x78)[0x56488bb6f9d8]
[gadi-cpu-clx-2095:102062] [14] python3(+0x1f2a45)[0x56488bb6fa45]
[gadi-cpu-clx-2095:102062] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x56488bac65da]
[gadi-cpu-clx-2095:102062] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x56488bb4545a]
[gadi-cpu-clx-2095:102062] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x56488ba96505]
[gadi-cpu-clx-2095:102062] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x56488bab6133]
[gadi-cpu-clx-2095:102062] [19] python3(+0x181455)[0x56488bafe455]
[gadi-cpu-clx-2095:102062] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x56488bb44762]
[gadi-cpu-clx-2095:102062] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x56488bab5d37]
[gadi-cpu-clx-2095:102062] [22] python3(+0x181455)[0x56488bafe455]
[gadi-cpu-clx-2095:102062] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x56488bb404d1]
[gadi-cpu-clx-2095:102062] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x56488bab5d37]
[gadi-cpu-clx-2095:102062] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x56488bb402b5]
[gadi-cpu-clx-2095:102062] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x56488bab5d37]
[gadi-cpu-clx-2095:102062] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x56488bb402b5]
[gadi-cpu-clx-2095:102062] [28] python3(_PyObject_FastCallDict+0x1b6)[0x56488ba97ab6]
[gadi-cpu-clx-2095:102062] [29] python3(+0x11b9a2)[0x56488ba989a2]
[gadi-cpu-clx-2095:102062] *** End of error message ***
[gadi-cpu-clx-2095:102067] *** Process received signal ***
[gadi-cpu-clx-2095:102067] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102067] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102067] Failing at address: 0x14d1c4070008
[gadi-cpu-clx-2095:102067] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x14d1e3a96dd0]
[gadi-cpu-clx-2095:102067] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x14d1ba8390b9]
[gadi-cpu-clx-2095:102067] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x14d1ba62f2a3]
[gadi-cpu-clx-2095:102067] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x14d1ba838d54]
[gadi-cpu-clx-2095:102067] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x14d1bb98461e]
[gadi-cpu-clx-2095:102067] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x14d1bb93ba5d]
[gadi-cpu-clx-2095:102067] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x14d1bb8f83a4]
[gadi-cpu-clx-2095:102067] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x14d1bbbd47fb]
[gadi-cpu-clx-2095:102067] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x14d1c02e098e]
[gadi-cpu-clx-2095:102067] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x14d1c203b2da]
[gadi-cpu-clx-2095:102067] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x14d1c232a4b2]
[gadi-cpu-clx-2095:102067] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x14d1c234bd79]
[gadi-cpu-clx-2095:102067] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x14d1c260da5c]
[gadi-cpu-clx-2095:102067] [13] python3(PyModule_ExecDef+0x78)[0x557b2229f9d8]
[gadi-cpu-clx-2095:102067] [14] python3(+0x1f2a45)[0x557b2229fa45]
[gadi-cpu-clx-2095:102067] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x557b221f65da]
[gadi-cpu-clx-2095:102067] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x557b2227545a]
[gadi-cpu-clx-2095:102067] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x557b221c6505]
[gadi-cpu-clx-2095:102067] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x557b221e6133]
[gadi-cpu-clx-2095:102067] [19] python3(+0x181455)[0x557b2222e455]
[gadi-cpu-clx-2095:102067] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x557b22274762]
[gadi-cpu-clx-2095:102067] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x557b221e5d37]
[gadi-cpu-clx-2095:102067] [22] python3(+0x181455)[0x557b2222e455]
[gadi-cpu-clx-2095:102067] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x557b222704d1]
[gadi-cpu-clx-2095:102067] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x557b221e5d37]
[gadi-cpu-clx-2095:102067] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x557b222702b5]
[gadi-cpu-clx-2095:102067] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x557b221e5d37]
[gadi-cpu-clx-2095:102067] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x557b222702b5]
[gadi-cpu-clx-2095:102067] [28] python3(_PyObject_FastCallDict+0x1b6)[0x557b221c7ab6]
[gadi-cpu-clx-2095:102067] [29] python3(+0x11b9a2)[0x557b221c89a2]
[gadi-cpu-clx-2095:102067] *** End of error message ***
[gadi-cpu-clx-2095:102068] *** Process received signal ***
[gadi-cpu-clx-2095:102068] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102068] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102068] Failing at address: 0x155461d0a008
[gadi-cpu-clx-2095:102068] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x155481730dd0]
[gadi-cpu-clx-2095:102068] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x15545c5b10b9]
[gadi-cpu-clx-2095:102068] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x15545c3a72a3]
[gadi-cpu-clx-2095:102068] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x15545c5b0d54]
[gadi-cpu-clx-2095:102068] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x15545d6fc61e]
[gadi-cpu-clx-2095:102068] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x15545d6b3a5d]
[gadi-cpu-clx-2095:102068] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x15545d6703a4]
[gadi-cpu-clx-2095:102068] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x15545d94c7fb]
[gadi-cpu-clx-2095:102068] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x15545df7a98e]
[gadi-cpu-clx-2095:102068] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x15545fcd52da]
[gadi-cpu-clx-2095:102068] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x15545ffc44b2]
[gadi-cpu-clx-2095:102068] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x15545ffe5d79]
[gadi-cpu-clx-2095:102068] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x1554602a7a5c]
[gadi-cpu-clx-2095:102068] [13] python3(PyModule_ExecDef+0x78)[0x5572fe4619d8]
[gadi-cpu-clx-2095:102068] [14] python3(+0x1f2a45)[0x5572fe461a45]
[gadi-cpu-clx-2095:102068] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x5572fe3b85da]
[gadi-cpu-clx-2095:102068] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x5572fe43745a]
[gadi-cpu-clx-2095:102068] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x5572fe388505]
[gadi-cpu-clx-2095:102068] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x5572fe3a8133]
[gadi-cpu-clx-2095:102068] [19] python3(+0x181455)[0x5572fe3f0455]
[gadi-cpu-clx-2095:102068] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x5572fe436762]
[gadi-cpu-clx-2095:102068] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x5572fe3a7d37]
[gadi-cpu-clx-2095:102068] [22] python3(+0x181455)[0x5572fe3f0455]
[gadi-cpu-clx-2095:102068] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x5572fe4324d1]
[gadi-cpu-clx-2095:102068] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x5572fe3a7d37]
[gadi-cpu-clx-2095:102068] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x5572fe4322b5]
[gadi-cpu-clx-2095:102068] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x5572fe3a7d37]
[gadi-cpu-clx-2095:102068] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x5572fe4322b5]
[gadi-cpu-clx-2095:102068] [28] python3(_PyObject_FastCallDict+0x1b6)[0x5572fe389ab6]
[gadi-cpu-clx-2095:102068] [29] python3(+0x11b9a2)[0x5572fe38a9a2]
[gadi-cpu-clx-2095:102068] *** End of error message ***
[gadi-cpu-clx-2095:102071] *** Process received signal ***
[gadi-cpu-clx-2095:102071] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102071] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102071] Failing at address: 0x14b9bc115008
[gadi-cpu-clx-2095:102071] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x14b9dbb3bdd0]
[gadi-cpu-clx-2095:102071] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x14b9b69bc0b9]
[gadi-cpu-clx-2095:102071] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x14b9b67b22a3]
[gadi-cpu-clx-2095:102071] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x14b9b69bbd54]
[gadi-cpu-clx-2095:102071] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x14b9b7b0761e]
[gadi-cpu-clx-2095:102071] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x14b9b7abea5d]
[gadi-cpu-clx-2095:102071] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x14b9b7a7b3a4]
[gadi-cpu-clx-2095:102071] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x14b9b7d577fb]
[gadi-cpu-clx-2095:102071] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x14b9b838598e]
[gadi-cpu-clx-2095:102071] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x14b9ba0e02da]
[gadi-cpu-clx-2095:102071] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x14b9ba3cf4b2]
[gadi-cpu-clx-2095:102071] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x14b9ba3f0d79]
[gadi-cpu-clx-2095:102071] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x14b9ba6b2a5c]
[gadi-cpu-clx-2095:102071] [13] python3(PyModule_ExecDef+0x78)[0x562953a999d8]
[gadi-cpu-clx-2095:102071] [14] python3(+0x1f2a45)[0x562953a99a45]
[gadi-cpu-clx-2095:102071] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x5629539f05da]
[gadi-cpu-clx-2095:102071] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x562953a6f45a]
[gadi-cpu-clx-2095:102071] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x5629539c0505]
[gadi-cpu-clx-2095:102071] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x5629539e0133]
[gadi-cpu-clx-2095:102071] [19] python3(+0x181455)[0x562953a28455]
[gadi-cpu-clx-2095:102071] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x562953a6e762]
[gadi-cpu-clx-2095:102071] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x5629539dfd37]
[gadi-cpu-clx-2095:102071] [22] python3(+0x181455)[0x562953a28455]
[gadi-cpu-clx-2095:102071] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x562953a6a4d1]
[gadi-cpu-clx-2095:102071] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x5629539dfd37]
[gadi-cpu-clx-2095:102071] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x562953a6a2b5]
[gadi-cpu-clx-2095:102071] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x5629539dfd37]
[gadi-cpu-clx-2095:102071] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x562953a6a2b5]
[gadi-cpu-clx-2095:102071] [28] python3(_PyObject_FastCallDict+0x1b6)[0x5629539c1ab6]
[gadi-cpu-clx-2095:102071] [29] python3(+0x11b9a2)[0x5629539c29a2]
[gadi-cpu-clx-2095:102071] *** End of error message ***
[gadi-cpu-clx-2095:102066] *** Process received signal ***
[gadi-cpu-clx-2095:102066] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102066] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102066] Failing at address: 0x14acf6345008
[gadi-cpu-clx-2095:102066] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x14ad15d6bdd0]
[gadi-cpu-clx-2095:102066] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x14acf0bec0b9]
[gadi-cpu-clx-2095:102066] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x14acf09e22a3]
[gadi-cpu-clx-2095:102066] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x14acf0bebd54]
[gadi-cpu-clx-2095:102066] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x14acf1d3761e]
[gadi-cpu-clx-2095:102066] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x14acf1ceea5d]
[gadi-cpu-clx-2095:102066] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x14acf1cab3a4]
[gadi-cpu-clx-2095:102066] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x14acf1f877fb]
[gadi-cpu-clx-2095:102066] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x14acf25b598e]
[gadi-cpu-clx-2095:102066] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x14acf43102da]
[gadi-cpu-clx-2095:102066] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x14acf45ff4b2]
[gadi-cpu-clx-2095:102066] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x14acf4620d79]
[gadi-cpu-clx-2095:102066] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x14acf48e2a5c]
[gadi-cpu-clx-2095:102066] [13] python3(PyModule_ExecDef+0x78)[0x557408d219d8]
[gadi-cpu-clx-2095:102066] [14] python3(+0x1f2a45)[0x557408d21a45]
[gadi-cpu-clx-2095:102066] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x557408c785da]
[gadi-cpu-clx-2095:102066] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x557408cf745a]
[gadi-cpu-clx-2095:102066] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x557408c48505]
[gadi-cpu-clx-2095:102066] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x557408c68133]
[gadi-cpu-clx-2095:102066] [19] python3(+0x181455)[0x557408cb0455]
[gadi-cpu-clx-2095:102066] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x557408cf6762]
[gadi-cpu-clx-2095:102066] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x557408c67d37]
[gadi-cpu-clx-2095:102066] [22] python3(+0x181455)[0x557408cb0455]
[gadi-cpu-clx-2095:102066] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x557408cf24d1]
[gadi-cpu-clx-2095:102066] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x557408c67d37]
[gadi-cpu-clx-2095:102066] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x557408cf22b5]
[gadi-cpu-clx-2095:102066] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x557408c67d37]
[gadi-cpu-clx-2095:102066] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x557408cf22b5]
[gadi-cpu-clx-2095:102066] [28] python3(_PyObject_FastCallDict+0x1b6)[0x557408c49ab6]
[gadi-cpu-clx-2095:102066] [29] python3(+0x11b9a2)[0x557408c4a9a2]
[gadi-cpu-clx-2095:102066] *** End of error message ***
[gadi-cpu-clx-2095:102070] *** Process received signal ***
[gadi-cpu-clx-2095:102070] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102070] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102070] Failing at address: 0x15517dc81008
[gadi-cpu-clx-2095:102070] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x15519d6a7dd0]
[gadi-cpu-clx-2095:102070] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x1551785280b9]
[gadi-cpu-clx-2095:102070] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x15517831e2a3]
[gadi-cpu-clx-2095:102070] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x155178527d54]
[gadi-cpu-clx-2095:102070] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x15517967361e]
[gadi-cpu-clx-2095:102070] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x15517962aa5d]
[gadi-cpu-clx-2095:102070] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x1551795e73a4]
[gadi-cpu-clx-2095:102070] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x1551798c37fb]
[gadi-cpu-clx-2095:102070] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x155179ef198e]
[gadi-cpu-clx-2095:102070] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x15517bc8c2da]
[gadi-cpu-clx-2095:102070] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x15517bf7b4b2]
[gadi-cpu-clx-2095:102070] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x15517bf9cd79]
[gadi-cpu-clx-2095:102070] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x15517c25ea5c]
[gadi-cpu-clx-2095:102070] [13] python3(PyModule_ExecDef+0x78)[0x55f42242a9d8]
[gadi-cpu-clx-2095:102070] [14] python3(+0x1f2a45)[0x55f42242aa45]
[gadi-cpu-clx-2095:102070] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x55f4223815da]
[gadi-cpu-clx-2095:102070] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x55f42240045a]
[gadi-cpu-clx-2095:102070] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x55f422351505]
[gadi-cpu-clx-2095:102070] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x55f422371133]
[gadi-cpu-clx-2095:102070] [19] python3(+0x181455)[0x55f4223b9455]
[gadi-cpu-clx-2095:102070] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x55f4223ff762]
[gadi-cpu-clx-2095:102070] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x55f422370d37]
[gadi-cpu-clx-2095:102070] [22] python3(+0x181455)[0x55f4223b9455]
[gadi-cpu-clx-2095:102070] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x55f4223fb4d1]
[gadi-cpu-clx-2095:102070] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x55f422370d37]
[gadi-cpu-clx-2095:102070] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x55f4223fb2b5]
[gadi-cpu-clx-2095:102070] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x55f422370d37]
[gadi-cpu-clx-2095:102070] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x55f4223fb2b5]
[gadi-cpu-clx-2095:102070] [28] python3(_PyObject_FastCallDict+0x1b6)[0x55f422352ab6]
[gadi-cpu-clx-2095:102070] [29] python3(+0x11b9a2)[0x55f4223539a2]
[gadi-cpu-clx-2095:102070] *** End of error message ***
[gadi-cpu-clx-2095:102073] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x1458f92d7dd0]
[gadi-cpu-clx-2095:102073] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x1458d41580b9]
[gadi-cpu-clx-2095:102073] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x1458cfdf82a3]
[gadi-cpu-clx-2095:102073] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x1458d4157d54]
[gadi-cpu-clx-2095:102073] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x1458d52a361e]
[gadi-cpu-clx-2095:102073] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x1458d525aa5d]
[gadi-cpu-clx-2095:102073] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x1458d52173a4]
[gadi-cpu-clx-2095:102073] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x1458d54f37fb]
[gadi-cpu-clx-2095:102073] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x1458d5b2198e]
[gadi-cpu-clx-2095:102073] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x1458d787c2da]
[gadi-cpu-clx-2095:102073] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x1458d7b6b4b2]
[gadi-cpu-clx-2095:102073] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x1458d7b8cd79]
[gadi-cpu-clx-2095:102073] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x1458d7e4ea5c]
[gadi-cpu-clx-2095:102073] [13] python3(PyModule_ExecDef+0x78)[0x55eab9a839d8]
[gadi-cpu-clx-2095:102073] [14] python3(+0x1f2a45)[0x55eab9a83a45]
[gadi-cpu-clx-2095:102073] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x55eab99da5da]
[gadi-cpu-clx-2095:102073] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x55eab9a5945a]
[gadi-cpu-clx-2095:102073] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x55eab99aa505]
[gadi-cpu-clx-2095:102073] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x55eab99ca133]
[gadi-cpu-clx-2095:102073] [19] python3(+0x181455)[0x55eab9a12455]
[gadi-cpu-clx-2095:102073] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x55eab9a58762]
[gadi-cpu-clx-2095:102073] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x55eab99c9d37]
[gadi-cpu-clx-2095:102073] [22] python3(+0x181455)[0x55eab9a12455]
[gadi-cpu-clx-2095:102073] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x55eab9a544d1]
[gadi-cpu-clx-2095:102073] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x55eab99c9d37]
[gadi-cpu-clx-2095:102073] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x55eab9a542b5]
[gadi-cpu-clx-2095:102073] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x55eab99c9d37]
[gadi-cpu-clx-2095:102073] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x55eab9a542b5]
[gadi-cpu-clx-2095:102073] [28] python3(_PyObject_FastCallDict+0x1b6)[0x55eab99abab6]
[gadi-cpu-clx-2095:102073] [29] python3(+0x11b9a2)[0x55eab99ac9a2]
[gadi-cpu-clx-2095:102073] *** End of error message ***
[gadi-cpu-clx-2095:102069] *** Process received signal ***
[gadi-cpu-clx-2095:102069] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102069] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102069] Failing at address: 0x153545f3a008
[gadi-cpu-clx-2095:102069] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x153565960dd0]
[gadi-cpu-clx-2095:102069] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x1535407e10b9]
[gadi-cpu-clx-2095:102069] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x1535405d72a3]
[gadi-cpu-clx-2095:102069] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x1535407e0d54]
[gadi-cpu-clx-2095:102069] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x15354192c61e]
[gadi-cpu-clx-2095:102069] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x1535418e3a5d]
[gadi-cpu-clx-2095:102069] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x1535418a03a4]
[gadi-cpu-clx-2095:102069] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x153541b7c7fb]
[gadi-cpu-clx-2095:102069] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x1535421aa98e]
[gadi-cpu-clx-2095:102069] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x153543f052da]
[gadi-cpu-clx-2095:102069] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x1535441f44b2]
[gadi-cpu-clx-2095:102069] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x153544215d79]
[gadi-cpu-clx-2095:102069] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x1535444d7a5c]
[gadi-cpu-clx-2095:102069] [13] python3(PyModule_ExecDef+0x78)[0x55b2b58599d8]
[gadi-cpu-clx-2095:102069] [14] python3(+0x1f2a45)[0x55b2b5859a45]
[gadi-cpu-clx-2095:102069] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x55b2b57b05da]
[gadi-cpu-clx-2095:102069] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x55b2b582f45a]
[gadi-cpu-clx-2095:102069] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x55b2b5780505]
[gadi-cpu-clx-2095:102069] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x55b2b57a0133]
[gadi-cpu-clx-2095:102069] [19] python3(+0x181455)[0x55b2b57e8455]
[gadi-cpu-clx-2095:102069] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x55b2b582e762]
[gadi-cpu-clx-2095:102069] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x55b2b579fd37]
[gadi-cpu-clx-2095:102069] [22] python3(+0x181455)[0x55b2b57e8455]
[gadi-cpu-clx-2095:102069] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x55b2b582a4d1]
[gadi-cpu-clx-2095:102069] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x55b2b579fd37]
[gadi-cpu-clx-2095:102069] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x55b2b582a2b5]
[gadi-cpu-clx-2095:102069] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x55b2b579fd37]
[gadi-cpu-clx-2095:102069] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x55b2b582a2b5]
[gadi-cpu-clx-2095:102069] [28] python3(_PyObject_FastCallDict+0x1b6)[0x55b2b5781ab6]
[gadi-cpu-clx-2095:102069] [29] python3(+0x11b9a2)[0x55b2b57829a2]
[gadi-cpu-clx-2095:102069] *** End of error message ***
[gadi-cpu-clx-2095:102074] *** Process received signal ***
[gadi-cpu-clx-2095:102074] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102074] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102074] Failing at address: 0x14c5c8ab8008
[gadi-cpu-clx-2095:102074] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x14c5e84dedd0]
[gadi-cpu-clx-2095:102074] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x14c5bf1a90b9]
[gadi-cpu-clx-2095:102074] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x14c5bef9f2a3]
[gadi-cpu-clx-2095:102074] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x14c5bf1a8d54]
[gadi-cpu-clx-2095:102074] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x14c5c44aa61e]
[gadi-cpu-clx-2095:102074] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x14c5c4461a5d]
[gadi-cpu-clx-2095:102074] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x14c5c441e3a4]
[gadi-cpu-clx-2095:102074] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x14c5c46fa7fb]
[gadi-cpu-clx-2095:102074] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x14c5c4d2898e]
[gadi-cpu-clx-2095:102074] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x14c5c6a832da]
[gadi-cpu-clx-2095:102074] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x14c5c6d724b2]
[gadi-cpu-clx-2095:102074] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x14c5c6d93d79]
[gadi-cpu-clx-2095:102074] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x14c5c7055a5c]
[gadi-cpu-clx-2095:102074] [13] python3(PyModule_ExecDef+0x78)[0x562673c329d8]
[gadi-cpu-clx-2095:102074] [14] python3(+0x1f2a45)[0x562673c32a45]
[gadi-cpu-clx-2095:102074] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x562673b895da]
[gadi-cpu-clx-2095:102074] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x562673c0845a]
[gadi-cpu-clx-2095:102074] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x562673b59505]
[gadi-cpu-clx-2095:102074] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x562673b79133]
[gadi-cpu-clx-2095:102074] [19] python3(+0x181455)[0x562673bc1455]
[gadi-cpu-clx-2095:102074] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x562673c07762]
[gadi-cpu-clx-2095:102074] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x562673b78d37]
[gadi-cpu-clx-2095:102074] [22] python3(+0x181455)[0x562673bc1455]
[gadi-cpu-clx-2095:102074] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x562673c034d1]
[gadi-cpu-clx-2095:102074] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x562673b78d37]
[gadi-cpu-clx-2095:102074] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x562673c032b5]
[gadi-cpu-clx-2095:102074] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x562673b78d37]
[gadi-cpu-clx-2095:102074] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x562673c032b5]
[gadi-cpu-clx-2095:102074] [28] python3(_PyObject_FastCallDict+0x1b6)[0x562673b5aab6]
[gadi-cpu-clx-2095:102074] [29] python3(+0x11b9a2)[0x562673b5b9a2]
[gadi-cpu-clx-2095:102074] *** End of error message ***
[gadi-cpu-clx-2095:102076] *** Process received signal ***
[gadi-cpu-clx-2095:102076] Signal: Segmentation fault (11)
[gadi-cpu-clx-2095:102076] Signal code: Address not mapped (1)
[gadi-cpu-clx-2095:102076] Failing at address: 0x147d712b8008
[gadi-cpu-clx-2095:102076] [ 0] /lib64/libpthread.so.0(+0x12dd0)[0x147d90cdedd0]
[gadi-cpu-clx-2095:102076] [ 1] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(pmix_gds_ds21_lock_init+0x129)[0x147d679e10b9]
[gadi-cpu-clx-2095:102076] [ 2] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmca_common_dstore.so.1(pmix_common_dstor_init+0x9a3)[0x147d677d72a3]
[gadi-cpu-clx-2095:102076] [ 3] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/pmix/mca_gds_ds21.so(+0x1d54)[0x147d679e0d54]
[gadi-cpu-clx-2095:102076] [ 4] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_gds_base_select+0x11e)[0x147d6ccaa61e]
[gadi-cpu-clx-2095:102076] [ 5] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_pmix_rte_init+0x88d)[0x147d6cc61a5d]
[gadi-cpu-clx-2095:102076] [ 6] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x147d6cc1e3a4]
[gadi-cpu-clx-2095:102076] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x147d6cefa7fb]
[gadi-cpu-clx-2095:102076] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x147d6d52898e]
[gadi-cpu-clx-2095:102076] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x147d6f2c32da]
[gadi-cpu-clx-2095:102076] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x147d6f5b24b2]
[gadi-cpu-clx-2095:102076] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x147d6f5d3d79]
[gadi-cpu-clx-2095:102076] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x147d6f895a5c]
[gadi-cpu-clx-2095:102076] [13] python3(PyModule_ExecDef+0x78)[0x559d405dd9d8]
[gadi-cpu-clx-2095:102076] [14] python3(+0x1f2a45)[0x559d405dda45]
[gadi-cpu-clx-2095:102076] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x559d405345da]
[gadi-cpu-clx-2095:102076] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x559d405b345a]
[gadi-cpu-clx-2095:102076] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x559d40504505]
[gadi-cpu-clx-2095:102076] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x559d40524133]
[gadi-cpu-clx-2095:102076] [19] python3(+0x181455)[0x559d4056c455]
[gadi-cpu-clx-2095:102076] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x559d405b2762]
[gadi-cpu-clx-2095:102076] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x559d40523d37]
[gadi-cpu-clx-2095:102076] [22] python3(+0x181455)[0x559d4056c455]
[gadi-cpu-clx-2095:102076] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x559d405ae4d1]
[gadi-cpu-clx-2095:102076] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x559d40523d37]
[gadi-cpu-clx-2095:102076] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x559d405ae2b5]
[gadi-cpu-clx-2095:102076] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x559d40523d37]
[gadi-cpu-clx-2095:102076] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x559d405ae2b5]
[gadi-cpu-clx-2095:102076] [28] python3(_PyObject_FastCallDict+0x1b6)[0x559d40505ab6]
[gadi-cpu-clx-2095:102076] [29] python3(+0x11b9a2)[0x559d405069a2]
[gadi-cpu-clx-2095:102076] *** End of error message ***
/apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libpmix.so.2(OPAL_MCA_PMIX3X_PMIx_Init+0xc4)[0x152edfd353a4]
[gadi-cpu-clx-2095:102075] [ 7] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_pmix_pmix3x.so(pmix3x_client_init+0xbb)[0x152ee40fd7fb]
[gadi-cpu-clx-2095:102075] [ 8] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/openmpi/mca_ess_pmi.so(+0x198e)[0x152ee472b98e]
[gadi-cpu-clx-2095:102075] [ 9] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libopen-rte.so.40(orte_init+0x29a)[0x152ee64c62da]
[gadi-cpu-clx-2095:102075] [10] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(ompi_mpi_init+0x232)[0x152ee67b54b2]
[gadi-cpu-clx-2095:102075] [11] /apps/openmpi-mofed4.7-pbs19.2/4.0.1/lib/libmpi.so.40(PMPI_Init_thread+0x79)[0x152ee67d6d79]
[gadi-cpu-clx-2095:102075] [12] /g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.04/lib/python3.7/site-packages/mpi4py/MPI.cpython-37m-x86_64-linux-gnu.so(+0x2fa5c)[0x152ee6a98a5c]
[gadi-cpu-clx-2095:102075] [13] python3(PyModule_ExecDef+0x78)[0x55fd05c419d8]
[gadi-cpu-clx-2095:102075] [14] python3(+0x1f2a45)[0x55fd05c41a45]
[gadi-cpu-clx-2095:102075] [15] python3(_PyMethodDef_RawFastCallDict+0x2da)[0x55fd05b985da]
[gadi-cpu-clx-2095:102075] [16] python3(_PyEval_EvalFrameDefault+0x559a)[0x55fd05c1745a]
[gadi-cpu-clx-2095:102075] [17] python3(_PyEval_EvalCodeWithName+0x255)[0x55fd05b68505]
[gadi-cpu-clx-2095:102075] [18] python3(_PyFunction_FastCallKeywords+0x583)[0x55fd05b88133]
[gadi-cpu-clx-2095:102075] [19] python3(+0x181455)[0x55fd05bd0455]
[gadi-cpu-clx-2095:102075] [20] python3(_PyEval_EvalFrameDefault+0x48a2)[0x55fd05c16762]
[gadi-cpu-clx-2095:102075] [21] python3(_PyFunction_FastCallKeywords+0x187)[0x55fd05b87d37]
[gadi-cpu-clx-2095:102075] [22] python3(+0x181455)[0x55fd05bd0455]
[gadi-cpu-clx-2095:102075] [23] python3(_PyEval_EvalFrameDefault+0x611)[0x55fd05c124d1]
[gadi-cpu-clx-2095:102075] [24] python3(_PyFunction_FastCallKeywords+0x187)[0x55fd05b87d37]
[gadi-cpu-clx-2095:102075] [25] python3(_PyEval_EvalFrameDefault+0x3f5)[0x55fd05c122b5]
[gadi-cpu-clx-2095:102075] [26] python3(_PyFunction_FastCallKeywords+0x187)[0x55fd05b87d37]
[gadi-cpu-clx-2095:102075] [27] python3(_PyEval_EvalFrameDefault+0x3f5)[0x55fd05c122b5]
[gadi-cpu-clx-2095:102075] [28] python3(_PyObject_FastCallDict+0x1b6)[0x55fd05b69ab6]
[gadi-cpu-clx-2095:102075] [29] python3(+0x11b9a2)[0x55fd05b6a9a2]
[gadi-cpu-clx-2095:102075] *** End of error message ***
--------------------------------------------------------------------------
mpirun noticed that process rank 19 with PID 0 on node gadi-cpu-clx-2095 exited on signal 11 (Segmentation fault).
--------------------------------------------------------------------------
